(Reviewed) Colosseum Translation

HotHeadEmperor opened the pull request.

HotHeadEmperor committed:
Update Colosseum.txt

HotHeadEmperor committed:
Update Colosseum.txt

HotHeadEmperor committed:
Update Colosseum.txt

HotHeadEmperor committed:
Translate 10039 (Cobra Girl)

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
-僕もカードマスターになりたいんです！\n握手してください！{主人公}選手みたいになれるように頑張ります！,
+今は先にギルドに行って昇格の手続きを終わらせよう（クエストメニューから昇格手続きが行えます）。,I should go to the guild and complete the promotion procedure first (you can do the promotion procedure from the quest menu).
+今は先にクエストを終わらせよう。,I should finish my quest first.
+ギルドで説明を受けた闘技場に、僕は足を運んでいた。,I made my way to the colosseum I had heard so much about at the guild.
```
<<Intro to the colosseum - lines 57-64, armored male silhouette speaking>>

I want to strike "so much" from this line.

The testing timeline was 'hear about the arena for the first time' -> 'go straight there', so it came across as the player character being a bit rude (like, "guild lady talked too much about this place")

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+再度話しかけていただければ、本日からでも対戦を組むことができますので、準備ができたらお声がけくださって構いませんよ。,If you speak to me again; we can arrange a match starting today; so feel free to let me know when you're ready.
+おや、あんた新人選手かい？\nそのリングの色、Dランクだね。,Oh hey; you're a rookie; aren't you? That ring color means you're a D-rank; right?
+闘技場では普段からカードマスターたちが試合をして\nその腕を競っているんだ。\n祭りの間はとくに試合が盛りだくさんさ。,In the colosseum; cardmasters usually compete in matches to showcase their skills. During the festival; matches here are especially common.
+国中から集まってきたカードマスターたちの火花散らす戦い…。\nいやぁ、たまらないね。,Cardmasters from all over the country gather and have fierce battles... Ah; the pull of it all is just irresistible; don't you think?
```
<<Speaker is "闘技場ファンの男,Male Colosseum Fan" lines 65-68>>

Alternate suggestion for the last sentence:

> Ah; the **excitement** of it all is just **intoxicating;** don't you think?

As a note, "Ah, I can't get enough of it." could also be a valid translation here, but I prefer the flow of your localization into the next line.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+祭りが始まる前まではサキュバスのカードなんて見たことがなかったんだけどなぁ。時代は変わったもんだ。,Before the festival started; I'd never even seen a succubus card. Times have really changed.
+この祭りが始まる少し前からかしら、\n闘技場での戦いがだいぶ様変わりしたのよね。,I guess it was just before this festival started that the battles in the colosseum changed quite a bit; wasn't it?
+前からエッチな攻撃をするカードマスターはいたのだけれど、\n急にサキュバスのモンスターを使うマスターが増えたというか…。,There were always cardmasters who used lewd attacks; but suddenly more and more masters started using succubus monsters...
+悪魔タイプ自体かなり珍しかったと思うし、\nなんでこんなにサキュバスモンスターが増えたのかしら。\n新しい遺跡で珍しいパックでも出たとか？,I think demon-types themselves were pretty rare before; so why have there been so many succubus monsters recently? Did they find a new rare pack out in the freshly-found ruins or something?
```
<<Speaker is "観客の女,Woman from the Audience" - lines 75-77>>

I don't like using "find" and "freshly-found ruins" in the same sentence, maybe replace with "new ruins"?
> Did they find a rare pack in the new ruins or something?

HotHeadEmperor replied:
changed 'the' to 'those' but otherwise borrowed word for word.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+もし使い道がなければ売ってしまってもかまいません。\nそのときはあなたが魔族領に行くときの、路銀にでも変えてください！,If can't really use it; feel free to sell it. If you did that; you could exchange it for travel expenses when you go to the demon realm!
+やぁやぁ、いらっしゃい。\nここでは闘技場の人気ランカーたちのグッズを売ってるよ。,Hey there; welcome! Here we sell merchandise from popular colosseum fighters!
+あの有名なモルゲン学園のエリートBランカー、メロディや\nシルヴィアのサイン色紙に・・・。,We have signed autograph boards from the famous elite B-rankers from Morgen Academy; Melody and Sylvia...
+モンスターを１枚も使わずに戦う、\n「百器使い」ムサシ選手の武器をモデルにした\n模造刀なんてのもあるよ。,There's even a replica sword modeled after the weapon of Musashi; the 'Hyakki Tsukai'; who fights without using a single monster!
```
<<Speaker is "グッズ売りの老婆【!】,Old Woman Selling Merchandise【!】" - Lines 277-296>>

'Hyakki Tsukai' is something like "Hundred Implements User"
"Master of a Hundred Implements" could fit, or if we cut it down then something like "Equipment Expert"

I think we could keep 'Hyakki Tsukai', but we'd have to add additional context elsewhere in the sentence.

(This one might be a bit kludgy, for illustrative purposes only 😅)
> There's even a replica sword modeled after the weapon of Musashi. She's called the 'Hyakki Tsukai' because she's a master of equipment that fights without using a single monster!

HotHeadEmperor replied:
Imma rope @Giratena on this one. I'm liking 'Equipment Master' most, but I think Head Translator's opinion is important here.

Giratena replied:
Hmm what about a combat title that has alliteration? Like, the "Equipment Expert", or "Toolkit Tyrant", "Hundred-Handled Harbinger"?

HotHeadEmperor replied:
What do you think of something simple, like 'Multi-weapon Master'?

Giratena replied:
I prefer the alliteration, so anything that uses it would be my preference ^^

HotHeadEmperor replied:
Well I'd argue that, since 'Multi-weapon' is one idea, it is alliteration, I'll give something else a try.

WisDZ reviewed (commented):
Group 1 reviewed!
Individual speaker lines for the first visit to the colosseum noted below in CSV (for convenience, I included it in the individual comments as well).

_Intro (lines 57 to 64)_
闘技場ファンの男,Male Colosseum Fan,(65-68)
観客の女,Woman from the Audience,(75-77)
エルル,Eruru,(358-361)
❤派手な女の子,❤Flashy Girl,(Separate card; not reviewed)
グッズ売りの老婆【!】,Old Woman Selling Merchandise【!】,(277-296)
学生の男,Male Student,(206-209)

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+いまよりずっと昔、闘技場が罪人の戦いを見る場所だったころ、\nそのころは罪人をなぶるのを見世物にして、\n勝者は敗者を好きにできる決まりになっていたそうなの。,Long ago; the colosseum was a place to watch criminals fight one another. It's said it was a spectacle to torment the criminals; and the victor had the right to do as they pleased with the vanquished.
+野蛮な風習がなくなった今でも、\nそのころの名残で闘技場の勝者が敗者を好きにできる習慣が\n残っているのだそうよ。,Even though the more barbaric customs have disappeared; it seems that remnants of that time still linger; allowing the winners of the colosseum to do as they please with the losers.
+ちなみに男のマスター同士の場合、魔力補給の名分がないから\n勝っても何かしたりできないの。\nそこは野蛮じゃなくていいわね。,By the way; when it's two male masters fighting; there's no pretext for mana replenishment; so even if you win; you can't really do anything to them. At least that part isn't so barbaric; right?
+なあ、アンタ、Dランクで最強のマスターって誰だと思う？\nやっぱり俺はミレーヌさんだね。,Hey; who do you think is the strongest master in D-rank? I think it's probably still Mylene; right?
```
<<Speaker is "観客の男,Man from the Audience" - lines 87-89>>

For the last sentence in line 87: 
Though we usually translate 'Ne' to an English equivalent at the end of the sentence, in this case I think we have a bit of leeway to combine the 'opinion seeking' aspect that it adds earlier on!

> If you ask me; it's gotta be Mylene.

Thoughts? (I only mention the change because I think it flows into the next line):
"I know she can't use high-level monsters; but those boobs... If I saw those; I'd lose all my fighting spirit right away!"

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+ちなみに男のマスター同士の場合、魔力補給の名分がないから\n勝っても何かしたりできないの。\nそこは野蛮じゃなくていいわね。,By the way; when it's two male masters fighting; there's no pretext for mana replenishment; so even if you win; you can't really do anything to them. At least that part isn't so barbaric; right?
+なあ、アンタ、Dランクで最強のマスターって誰だと思う？\nやっぱり俺はミレーヌさんだね。,Hey; who do you think is the strongest master in D-rank? I think it's probably still Mylene; right?
+いや、ミレーヌさんがレベルの高いモンスターを使えないのは\n俺も知ってるよ。でも、あのおっぱい。あんなの見たら、\n俺だったら即戦意を喪失しちゃうよ。,I know she can't use high-level monsters; but those boobs... If I saw those; I'd lose all my fighting spirit right away!
+実際、男相手の戦績はかなりいいみたいだしな。\nアンタも男のマスターだし、きっとミレーヌさんには勝てねえぜ。,Actually; it seems her track record against men is pretty impressive. So since you're a male master too; there's no way you'd win against Mylene.
```
Optional change:
> Actually; it seems her track record against men is pretty impressive. **You're** a male master too; **so** there's no way you'd win against Mylene.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+カードには色々な属性がありますが、\nDランク以上のマスターの場合、\n使用する属性を１つに限定しているマスターが多いみたいですね。,Cards have various attributes; but it seems that for masters ranked D and above; many tend to limit themselves to using just one; right?
+やはり属性サポートの強力なカードが使えるのが大きいのでしょう。\nしかし、属性に頼り切っていると、オクタ選手のような\n相手の属性を変えるカードに足元をすくわれることもあります。,It makes sense since being able to use powerful cards that support specific attributes is really helpful. If you rely too much on attributes; however; you might find yourself at a disadvantage against cards like the ones Octa uses; which can change the opponent's attribute.
+属性変化のカードは希少ですから使い手は多くありません。\nしかし、一応そういう妨害も頭に入れたうえでデッキを組むのは\n重要かもしれませんね。,Cards that change attributes are rare; so there aren't many masters who use them. Still; it might be useful to keep such disruptions in mind when building a deck; don't you think?
+「オーバーキル」という言葉を知っていますか？\n相手モンスターにとどめを刺す際に、\nHP以上にダメージを与えて倒すことを「オーバーキル」といいます。,Do you know the term "overkill"? It refers to dealing more damagae than the opponent has HP left to finish it off.
```
<<Speaker is "学生の男,Male Student" this is the second time meeting him, he reuses line 206 then says lines 210-213>>

Small typo: damage*

I want to mention: based on how the text is laid out, it seems like line 206 will be used a lot!
Line 206 reads: "Card battles are incredibly deep... There's so much to learn from the fights in the colosseum."
I'm a fan, it's a good signature line imo.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+{%%40327}を手に入れた！,I got {%%40327}!
+なんだい。一回勝ったぐらいで有名人気取りかい。\n気前のよかったアンタが、\nケチになっちゃうなんて、わたしゃ悲しいね。,What's up? Just because you won once; you think you're too famous for autographs now? It's sad to see someone who used to be so generous become so stingy; don't you think?
+何も買わないなら帰っておくれ。\nこの珍しいカードもほかの人に売っちまうよ。,If you're not going to buy anything; then just go home. I'll sell this rare card to someone else!
+おや、自分のサインがどれぐらい売れてるのか見に来たのかい？\nまあまだ一勝だからね。\n闘技場のマニアがちょっと買ってくぐらいだよ。,Oh; did you come to see how much your signature is selling for? Well; you've still only one once. Only a few colosseum enthusiasts buy it!
```
<<Speaker is "グッズ売りの老婆【!】,Old Woman Selling Merchandise【!】" this is the second time meeting her, she reuses line 277 then uses lines 297-308 depending on the situation>>

Small typo: you've still only won* once.

& side note: another signature line for this character it seems, line 277
Line 277 reads: "Hey there; welcome! Here we sell merchandise from popular colosseum fighters!"
Functional imo!

WisDZ reviewed (commented):
Group 2 reviewed!

盛り上がっている男,Excited Man,(81-82)
物知りな女性,Knowledgeable Woman,(83-86)
観客の男,Man from the Audience,(87-89)
グッズ売りの老婆【!】,Old Woman Selling Merchandise【!】,(277+297-308),2nd meeting
学生の男,Male Student,(206+210-213),2nd meeting

HotHeadEmperor committed:
Update Colosseum.txt

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+例えば相手のＨＰが800ポイントのとき、\n魔法カードで300ダメージを与え、その後攻撃力800のモンスターで\n攻撃したら、完全に魔法カードが１枚無駄になっています。,For example; if the opponent's HP is 800; and you deal 300 damage with a magic card before attacking with a monster that has 800 attack; then your magic card was a total waste!
+そんなバカなことはしないと思うかもしれませんが、\n相手モンスターが複数いたりすると、\nけっこうこの手のプレイングミスは起こりがちです。,You might think that no one would make such a silly mistake; but when your opponent has multiple monsters; these kinds of misplays can happen pretty easily.
+こちらの与えられるダメージと相手のＨＰを正確に把握し、\n攻撃手段を効率よく分配できるかどうかは、\n強いカードマスターになる鍵ですね。,Accurately understanding the damage you can deal and the opponent's HP so you can efficiently distribute your attack methods is key to becoming a strong cardmaster; don't you think?
+デッキ枚数を何枚にするか、というのは\nマスターにとって永遠のテーマです。\n一見、カードを入れるだけ入れた方eが強いように見えますが…,Deciding how many cards to include in your deck is an eternal question for a master to answer. At first glance; it seems that having more cards would make your deck stronger...
```
A loose "e" is in the JP side here, needs to be deleted.

WisDZ reviewed (commented):
Group 3 reviewed! (Only one change, and it's in the formatting)

ファンになった女性,Woman who Became a Fan,(90-93)
観客の男,Man from the Audience,(96-98),2nd meeting
酒くさい男,Man who Smells of Alcohol,(99-102)
グッズ売りの老婆【!】,Old Woman Selling Merchandise【!】,(277 & 309-319),3rd meeting
学生の男,Male Student,(214-217),3rd meeting

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+でもよ、今日は飲み友達同士で、誰が勝つか賭けてるんだ。\n俺はアンタに賭けたからよ、\n損させられた分、今日はもうけさせてくれよ！,But hey; today me and my drinking buddies are betting on who'll win. I bet on you; so make up for the loss and give me a win today!
+つまり、あんたには期待してるってこった！\nサキュバスにすら勝ったんだろ？なら次も楽勝だな！,I have high hopes for you! You even beat a succubus; right? So your next battle should be a piece of cake!
+この間Bランク同士の試合を見たんだけどよ、\nいやぁ、やっぱりすごいね。,The other day; I watched a match between two B-rank fighters; and wow; it was really something!
+最上級のモンスターがどんどん出てきての激しいぶつかり合い。\n１ターンにモンスターが２体も３体も出てくるし、\nやっぱり上位ランカーは下位とはわけが違うよ。,The top-tier monsters kept coming out; and the clashes were intense. In one turn; two or three massive monsters would get summoned. As expected; the higher-ranked players are on a different level than the lower-ranked ones.
```
<<Speaker is 満足そうな観客,Satisfied Audience Member (lines 103-105)>>

I kind of want to change "In one turn" to "In a single turn"
This bit as it is now ended up with a counting aesthetic to me: "...one turn, two or three..."

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+つまり、あんたには期待してるってこった！\nサキュバスにすら勝ったんだろ？なら次も楽勝だな！,I have high hopes for you! You even beat a succubus; right? So your next battle should be a piece of cake!
+この間Bランク同士の試合を見たんだけどよ、\nいやぁ、やっぱりすごいね。,The other day; I watched a match between two B-rank fighters; and wow; it was really something!
+最上級のモンスターがどんどん出てきての激しいぶつかり合い。\n１ターンにモンスターが２体も３体も出てくるし、\nやっぱり上位ランカーは下位とはわけが違うよ。,The top-tier monsters kept coming out; and the clashes were intense. In one turn; two or three massive monsters would get summoned. As expected; the higher-ranked players are on a different level than the lower-ranked ones.
+もう上の方のランクは女のマスターしかほぼ残ってないからね。\nエロいことは控えめでバトルもけっこうバチバチの戦いだし、\n見てて派手だったねー。高いチケットを買った甲斐があったな。,Almost all of the higher ranked masters are female these days. They keep the erotic stuff to a minimum when they're fighting eachother; and the battles get pretty intense. It was really flashy to watch. That expensive ticket was really worth the price.
```
<<Speaker is 満足そうな観客,Satisfied Audience Member (lines 103-105)>>

Microsoft word suggested "each other" rather than "eachother"

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+もう上の方のランクは女のマスターしかほぼ残ってないからね。\nエロいことは控えめでバトルもけっこうバチバチの戦いだし、\n見てて派手だったねー。高いチケットを買った甲斐があったな。,Almost all of the higher ranked masters are female these days. They keep the erotic stuff to a minimum when they're fighting eachother; and the battles get pretty intense. It was really flashy to watch. That expensive ticket was really worth the price.
+お、期待のルーキーじゃないか。\n俺、あんたのこの間の試合見たぜ。,Oh; aren't you that promising rookie? I watched your match the other day.
+同じ日に上位ランカーの試合があったから、\n実況もつかなかったみたいだけど、\nやっぱりあんた強いな。これで３連勝じゃないか。,Since there was a match with a top-ranked player on the same day; it seems there was no commentary for your fight; but you're really strong. That's three consecutive wins now; right?
+昔は男のマスターもAランクやBランクにいたんだけど、\nみんなここ数年でランクダウンしちまった。\nあんたは頑張って上のランクに行ってくれよ。楽しみにしてるぜ。,In the past; there were male masters in the A- and B-ranks; but all the guys have really dropped off over the past few years. Keep working hard to reach the higher ranks. I'm looking forward to it!
```
<<Speaker is 観客の男,Man from the Audience (lines 106-108)>>
Note: Though this is the third appearance of a character with this name, I believe it is a different 'Man from the Audience' each time.

Should we drop the hyphen after A?
"in the A and B-ranks"

HotHeadEmperor replied:
I actually found this rule for Gira in the past. When you list a series of things that would be hyphenated separately, you hang a hyphen on each

WisDZ replied:
Sounds good! Resolved

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+ううむ、男性マスターのグッズはあんまり売れないね。\n最近みんな降格しちゃって、\n人気がないから困っちゃうよ。,Hmm; merchandise for the male masters just doesn't sell well anymore. Recently; everyone of them's been demoted; and their lack of popularity is becoming a problem!
+・・・おや、また会ったね。\nあんたのサインは男マスターの中じゃ売れてるほうさ。これで\n３連勝だろう？あんたならもしかしたらって期待されてるのかもね。,...Oh; we meet again; eh? Your signature is one of the better-selling ones among the male masters. That's three wins in a row; right? Maybe they're expecting something from you; do you think?
+おお、ありがとう！\nいやぁさすがは男の中の男だね。\n婆に優しい、それだからこんなに強いんだ。,Oh; thank you! Wow; you're truly a man's man. You're kind to this old lady; that's why you're so strong. 
+ほらこれがお礼のカードだ。\n今度は正真正銘レアカードだよ。\nたぶん強いよ。,Here; take this thank-you card. It's a genuine rare card this time. It even lookes pretty strong!
```
<<Speaker is グッズ売りの老婆【!】,Old Woman Selling Merchandise【!】 (lines 277 + 320-328, depending on context), this is the fourth potential meeting>>

Small typo: looks*

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+{%%60036}を手に入れた！,I got {%%60036}!
+侍使いのマスターがわたしに売っていったんだ。\n爆乳マスターのキャミーに負けて、いまじゃすっかり落ちぶれちまったが、\n強くていいマスターだったんだよ。,The samurai-using master sold it to me. He lost to the busty master Cammy; and has completely fallen from grace now. Still; he was a good and strong master!
+男マスターの話をしてたらなんだか思い出しちまった。\nこんないいもの普段ならあげないんだけど、\nアンタなら上手く使いこなせるかもね。,Talking about the male masters reminded me I had it. I usually wouldn't give away something this good; but maybe you could make good use of it; right?
+なんだい。あんたは男マスターの中だと一番けちかもね。\n女マスターに勝てたからってそんなじゃすぐ負けちまうよ。,What is it? You might be the stingiest man among all the male masters. Even if you managed to beat a couple female masters; at this rate; you'll lose soon enough!
```
<<Speaker is グッズ売りの老婆【!】,Old Woman Selling Merchandise【!】 (lines 277 + 320-328, depending on context), this is the fourth potential meeting>>

This line occurs if you refuse to give a signature.
I think instead of "What is it?" as the first sentence, something more confrontational, "Well; well." or "What's the matter?" or even "Seriously?" might fit.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+「デッキケース」に入れて事前に魔力を充填しておかないと\nカードは連発して使えないので、\nいたしかたないところではありますね。,If you don't put the cards in the "deck case" and charge them with magical power in advance; you won't be able to use them repeatedly; so it's pretty unavoidable; huh?
+デッキケースは遺跡からよく発掘され、広く使われていますが、\n謎の多い魔道具です。入れたカードが自動でシャッフルされるのも、\n魔力の充填が効率よくなるからだと言われています。,Deck cases are often excavated from ruins and are widely used; but they're still a pretty mysterious magical tool. It's said that the cards placed inside are automatically shuffled because it makes charging them with magical power more efficient.
+強いカード10枚ぐらいでデッキを組めて、\nさらに事前に山札を並べておければ最高なのですが…。\nそううまい話はないようです。,If you could build a deck with about ten strong cards and maybe even arrange the draw pile in advance; you might be unstoppable... But it's just not that easy I guess.
+カードにはアドバンテージという考え方があります。\nアドバンテージというのは「どれぐらい有利になったか」\nみたいな意味ですね。,In battles; cards have a concept called 'advantage'. 'Card advantage' means 'how much more favorable a card makes your situation,' right?
```
<<Speaker is 学生の男,Male Student (lines 223-227) Fourth meeting>>

 Replace comma at the end with semicolon

WisDZ reviewed (commented):
Group 4!

満足そうな観客,Satisfied Audience Member,(103-105)
観客の男,Man from the Audience,(106-108),3rd meeting* (different individuals)
グッズ売りの老婆【!】,Old Woman Selling Merchandise【!】,(277+320-328),4th meeting
学生の男,Male Student,(223-227),4th meeting

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+お、期待のルーキーじゃないか。\n俺、あんたのこの間の試合見たぜ。,Oh; aren't you that promising rookie? I watched your match the other day.
+同じ日に上位ランカーの試合があったから、\n実況もつかなかったみたいだけど、\nやっぱりあんた強いな。これで３連勝じゃないか。,Since there was a match with a top-ranked player on the same day; it seems there was no commentary for your fight; but you're really strong. That's three consecutive wins now; right?
+昔は男のマスターもAランクやBランクにいたんだけど、\nみんなここ数年でランクダウンしちまった。\nあんたは頑張って上のランクに行ってくれよ。楽しみにしてるぜ。,In the past; there were male masters in the A- and B-ranks; but all the guys have really dropped off over the past few years. Keep working hard to reach the higher ranks. I'm looking forward to it!
+最近上位ランカーは女の選手ばっかりになったわ。\nだってみんな女モンスターに誘惑されたら\nすぐ負けちゃうんですもの。,Recently; the top-ranked players have all been female. It's all because; as soon as they get tempted by female monsters; the men lose immediately.
```
<<Speaker is 観客の女,Woman from the Audience (lines 109-111)>>

Suggestions for flow:
> **These days;** the top-ranked players **are all women.** It's because **guys just crumble** as soon as they get tempted by female monsters.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+ああ・・・❤またキャミー選手と戦いたい・・・❤,Ah...❤ I want to fight Cammy again...❤
+おっと、すまない。ぶつかるところだったね。\nきみもカードマスターかい？\n僕もそうさ。元はＣランクマスターだったんだぜ。,Oops; sorry. I almost bumped into you there. Are you a cardmaster? Yeah; me too. I used to be C-rank; actually.
+２週間前ぐらいになるんだが、\nCランクのキャミーという選手と戦ってね、\nその時の戦闘がどうにも忘れられないんだ。,It was about two weeks ago that I fought a C-rank player named Cammy; and I just can't get that battle out of my head.
+あの選手に負けたせいで\nCランクからDランクに落ちてしまったっていうのに、\n俺も自分で何を言ってるんだって思うよ。だけど・・・。,I dropped from C-rank to D-rank because of her; and I can't help believe I'm saying this... But...
```
<<Speaker is ふらふらのカードマスター,Dizzy Cardmaster (lines 112-117)>>

Extra word to drop:
> I dropped from C-rank to D-rank because of her; and I can't ~help~ believe I'm saying this... But...

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+おっと、すまない。ぶつかるところだったね。\nきみもカードマスターかい？\n僕もそうさ。元はＣランクマスターだったんだぜ。,Oops; sorry. I almost bumped into you there. Are you a cardmaster? Yeah; me too. I used to be C-rank; actually.
+２週間前ぐらいになるんだが、\nCランクのキャミーという選手と戦ってね、\nその時の戦闘がどうにも忘れられないんだ。,It was about two weeks ago that I fought a C-rank player named Cammy; and I just can't get that battle out of my head.
+あの選手に負けたせいで\nCランクからDランクに落ちてしまったっていうのに、\n俺も自分で何を言ってるんだって思うよ。だけど・・・。,I dropped from C-rank to D-rank because of her; and I can't help believe I'm saying this... But...
+キャミー選手の胸は\nほんとに脳がとけてしまうんじゃないかってぐらい気持ちよくて、\nそのうえ快楽攻撃ばっかり執拗にしてくるから…❤ううっ…❤,Cammy's chest just felt so good that it almost maked my brain melt; and on top of that; she attacks with pleasure nonstop...❤ Ugh...❤
```
<<Speaker is ふらふらのカードマスター,Dizzy Cardmaster (lines 112-117)>>

Likely typo: maked -> made*
Though it could be a choice for this particular character, we'd have to adjust the rest of the lines.

Also a recommendation from the translation bot. It broke the line into two sentences and I kind of like it:
> Cammy's breasts felt so good; it's like my brain was melting. And she just kept relentlessly hitting me with those pleasure attacks…❤ Oooh…❤

HotHeadEmperor replied:
Took your suggestion, but changed 'ohh' back to 'ughh' for a more frustrated groan.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+だからほらよ、持ってきな！\nこの金で新しいパックでも買って、\n上のランクのやつらをぎゃふんと言わせてやりな！,So here; take this! Use that to buy a new pack and make those higher-ranked girls eat their words!
+アンタのすげー試合を見せてもらえるなら、\nこれぐらい安いもんだぜ。\n楽しみにしてるぜ、ルーキー！,It's a small price to pay to get to see your amazing matches. I'm looking forward to it; rookie!
+よお、期待のルーキー！\n新しいカードは仕入れてきたか？,Hey; promising rookie! Did you get any new cards?
+なあに、アンタのすげー試合を見せてもらえるなら、\nこれぐらい安いもんだ。\n楽しみにしてるぜ、ルーキー！,Hey; it's a small price to pay to get to see your amazing matches. I'm looking forward to it; rookie!
```
<<Speaker is ファンの男【!】,Fanboy【!】 (lines 118-124)>>

I believe lines 123 and 124 are meant to be the 'revisit' lines for this interaction.*

I'd like to change one of them away from starting with "Hey" since they occur right after each other.
Line 123 to "Yo; ..."
or Line 124 to "Listen; ..."


*currently a bug on these lines, the 'revisit' lines play immediately after the first conversation. Opening issue.

WisDZ replied:
Linking issue: #333

WisDZ reviewed (commented):
Group 5!

観客の女,Woman from the Audience,(109-111),2nd meeting* (different individuals)
ふらふらのカードマスター,Dizzy Cardmaster,(112-117)
ファンの男【!】,Fanboy【!】,(118-124)
グッズ売りの老婆【!】,Old Woman Selling Merchandise【!】,(277+329-340),5th meeting
学生の男,Male Student,(228-232),5th meeting

Edit: Whoops! I pasted Group 4 to copy the formatting and forgot to delete it. Now removed.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -1,49 +1,49 @@
+ふらふらのカードマスター,Dizzy Cardmaster
+ファンの男【!】,Fanboy【!】
+ファンの男,Fanboy
+あこがれている少年,Admired Boy
```
This is probably meant to be 'Aspiring Boy' or 'Fan Boy'
He speaks lines 125-127, about how he wants to be as cool as you are, and then he says he won't wash his hand now that you've shaken it.

Giratena replied:
^ This! The boy isn't being admired. He's admiring the MC.

@HotHeadEmperor I think "Fan Boy" matches the best here.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+１対１交換をするカードにはこうした使い道もあるので、\n知っているとデッキ構築の幅が広がりますね。,Cards that can be exchanged one-for-one have these kinds of uses; so knowing this can broaden your deck-building options; don't you think?
+墓地リソースという言葉を知っていますか？,Do you know the term 'graveyard resource'?
+カードには自分の墓地のカードをコストにして発動するものがあります。\nまた墓地のカードを復活させたり、手札に戻すカードもあります。,Some cards can be activated using cards from your own graveyard as a cost. There are also cards that can revive cards from the graveyard or return them to your hand.
+そうしたカードのことを考えると、\n墓地もひとつの資源、重要なリソースだという考え方が、\n「墓地リソース」ですね。,Keeping those cards in mind; the idea that the graveyard is also a resource; and an important resource at that; is what we call 'graveyard resource', understand?
```
<<Speaker is 学生の男,Male Student (lines 233-237), 6th meeting>>

Need to sub comma at the end to a semicolon
I'd also like to break the line into two sentences as well to simplify it a bit.

Additional reword is a suggestion to localize the line differently - optional change:
> **If you use cards like those; the graveyard becomes another resource;** and an important resource at that. That is what we call 'graveyard resource'**;** understand?

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+墓地リソースという言葉を知っていますか？,Do you know the term 'graveyard resource'?
+カードには自分の墓地のカードをコストにして発動するものがあります。\nまた墓地のカードを復活させたり、手札に戻すカードもあります。,Some cards can be activated using cards from your own graveyard as a cost. There are also cards that can revive cards from the graveyard or return them to your hand.
+そうしたカードのことを考えると、\n墓地もひとつの資源、重要なリソースだという考え方が、\n「墓地リソース」ですね。,Keeping those cards in mind; the idea that the graveyard is also a resource; and an important resource at that; is what we call 'graveyard resource', understand?
+墓地をフル活用するデッキの使い手は\n「墓地は第二の手札だ」と豪語する人もいるとか。\nそれぐらい墓地は重要なのです。,Some users whose decks fully utilize the graveyard boast that 'their graveyard is a second hand'. That's how important the graveyard can be.
```
<<Speaker is 学生の男,Male Student (lines 233-237), 6th meeting>>

Should we swap 'users' to 'masters'?
使い手 can be 'user' 'employer' or 'master' (of a discipline)

(This one might just be me - I associate user with 'user of a program')

HotHeadEmperor replied:
I like it. I think I'll go with that.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+・・・おや、また会ったね。\nにわとりはちゃんと働いてるかい？\nバトルが終わったら産んだ卵も消えちまうらしいからそこが残念だね。,...Oh; we meet again; eh? Is that chicken I gave you working out? It seems that the eggs it lays disappear after battle. Such a shame; don't you think?
+まあそうじゃなければ\nカードでいくらでも何でも生み出せるからね。\nあたしらみたいな一般人は商売あがったりだったろうね。,I guess if they didn't; you could create just about anything and everything with cards. People like us; the common folk; would have been out of business; huh?
+一般人にも生きる道を与えてくれた女神さまに感謝だよ。\nできればカードを使える魔力の方がほしかったけどね。,I'm grateful to the goddess who gave ordinary people a way to live. If possible; I would have preferred a magic power that could use cards though; you know?
+Bランクマスターシルヴィア選手の最強カード、イージスの人形に、\nモルゲンのエース、Aランクマスターエイライラのハーピーの羽。\nどれも珍しいよ。,The strongest card of the B-rank master Sylvia; the Aegis Doll; and Morgen's ace; A Harpy Feather from the A-rank Master Eilira. Rare items for sale!
```
<<Speaker is グッズ売りの老婆,Old Woman Selling Merchandise (line 277 then 341-350 depending on context), 6th meeting>>

I'd like to break the first sentence into two and adjust for clarity:
> Here's the Aegis Doll; the strongest card of B-rank Master Sylvia! We also have a Harpy Feather from Morgen's ace; A-rank Master Eilaira! Rare items for sale!

WisDZ reviewed (commented):
Group 6!

あこがれている少年,Admired Boy,(125-127)
事情通の男,Knowledgeable Man,(128-130)
下位のカードマスター,Lower-Tier Cardmaster,(131-133)
グッズ売りの老婆【!】,Old Woman Selling Merchandise【!】,(277+341-350),6th meeting
学生の男,Male Student,(233-237),6th meeting

HotHeadEmperor committed:
Update Colosseum.txt

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+カードマスターが犯罪をしたらどうなると思う？\nギルドで指名手配されて、\nモルゲンの特殊牢獄に入れられるのよ。,What do you think happens when a cardmaster commits a crime? They end up wanted by the guild and locked up in Morgen's special prison.
+カードを取り上げたうえで、\n強いカードマスターの看守たちが24時間見張っているの。\n凶悪な犯罪者だって逃げだすことはできないわ。,They take away your cards and strong cardmaster guards watch over you 24/7. Even the most notorious criminals can't escape.
+看守の一人、Cランクのヤツメさまはとってもカッコいいんだから。\nあのクールな瞳。わたしあの人のファンなの。\n無敗のルーキーのあなたにだってきっとヤツメさまは負けないわよ。,One of the guards; a C-rank master named Yatsume; is really cool. Those intense eyes of hers... I'm a big fan. Even you wouldn't stand a chance against her; undefeated rookie!
+なあ聞いてくれよ。\n俺の応援してた男のCランクカードマスターがこの間ぼろ負けしてよ、\nついに降格しちまったんだ。,Hey; listen up! The C-rank cardmaster I was cheering for got completely crushed the other day and finally got demoted.
```
<<Speaker is 気落ちしてる男,Downcast Man (lines 137-140)>>

Instead of 'finally got demoted' I would rather use 'ended up getting demoted'
> Hey; listen up! The C-rank cardmaster I was cheering for got completely crushed the other day. **He ended up getting demoted.**

Since the speaker is 'Downcast man' "Hey; listen to this." might be better for a first sentence.
AI advice also suggested using "You won't believe what happened." in the above context.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+女どもの誘惑に負けないけっこう強いマスターだったんだぜ？\nだけど、一回の試合では我慢できても、\n何回も戦っているうちにとうとう耐え切れなくなったみたいでよ。,He was a pretty strong cardmaster who wasn't swayed by the women's temptations; you know? But while he might've been able to hold out in one match; it seems after back-to-back fights he just couldn't stand it anymore.
+一回こっぴどく負けてからはすっかり快楽に弱くなっちまって…。\nそこからはいくらやっても女マスターに勝てなくなっちまった。,After losing badly just once; he became totally weak to pleasure attacks... From then on; no matter how hard he tried; he couldn't win against the female masters anymore.
+一度でも女モンスターたちの気持ちよさ味わうと、\n快感に抗えなくなっちまうのかな。\nアンタは負けないでくれよ、無敗のルーキー。,Once you experience the pleasure of a female monster even once; you can't resist the ecstasy anymore. So please; undefeated rookie; don't lose.
+ネネコ選手みたいな獣人って、\n普段は北の方に住んでいる種族なのよ。\nだから南にあるモルゲンに来ると暑くて大変らしいわ。,Beastmen like Nenekko usually live in the northern regions; so when they come to Morgen in the south; it seems to be quite hot and tough for them.
```
<<Speaker is もふもふ好きの女,Woman who Loves Fluffy Things (lines 141 -142)>>

Neneko*

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+一回こっぴどく負けてからはすっかり快楽に弱くなっちまって…。\nそこからはいくらやっても女マスターに勝てなくなっちまった。,After losing badly just once; he became totally weak to pleasure attacks... From then on; no matter how hard he tried; he couldn't win against the female masters anymore.
+一度でも女モンスターたちの気持ちよさ味わうと、\n快感に抗えなくなっちまうのかな。\nアンタは負けないでくれよ、無敗のルーキー。,Once you experience the pleasure of a female monster even once; you can't resist the ecstasy anymore. So please; undefeated rookie; don't lose.
+ネネコ選手みたいな獣人って、\n普段は北の方に住んでいる種族なのよ。\nだから南にあるモルゲンに来ると暑くて大変らしいわ。,Beastmen like Nenekko usually live in the northern regions; so when they come to Morgen in the south; it seems to be quite hot and tough for them.
+毛がもふもふしてて、人間よりも体温高そうだもの。\nでも、獣人の男の子ってかわいいから、\n一度あのもふもふを思いっきりなでてみたいわね。,Their fur is really thick; and they seem to have a higher body temperature than humans. But beastmen boys are so cute; I want to try giving that fluff a good petting at least once; you know?
```
<<Speaker is もふもふ好きの女,Woman who Loves Fluffy Things (lines 141 -142)>>

Their fur is really fluffy*

After all, the もふもふ-loving woman would know her もふもふ for sure 😄

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+おや、なんだい、けちなルーキーさまじゃないか。\n気が変わったのかい。\nサインを書いてくれれば、この珍しいカードをあげるよ。,Oh; what's this? Aren't you the stingy rookie? Have you changed your mind? If you sign it; I'll give you this rare card!
+最近の売れ筋はこれ、あの無敗のルーキーのサインだよ。\n男の身なのに勝利に次ぐ勝利、\nCランクでも快進撃を続けてるあのルーキーだよ。,The recent bestseller is this; the signature of that undefeated rookie. Despite being a man; he keeps winning victory after victory; continuing his impressive run even into C-rank!
+おお、ありがとう！\nやっぱりアンタはいい男だね。優しいし強いし、\nいやぁモルゲンで一番男前かもしれないね。,Oh; thank you! You really are a good man. You're kind and strong; and you might just be the most handsome guy in Morgen; you know?
+ほらこれがお礼のカードだ。\n昔ネネコがいらないってくれたネズミのカードだよ。\nとげとげしてるから気に入らないとか言ってたね。,Here; this is a thank-you card. It's a mouse card that Nenekko didn't want and gave to me a long time ago. She said she didn't like it because it had I bad attitude; I suppose.
```
<<Speaker is グッズ売りの老婆,Old Woman Selling Merchandise (line 277 then lines 351-357 depending on context)>>

Alternate line suggestion:
> Here; this is a thank-you card. It's a mouse card that **Neneko** didn't want and gave to me a long time ago. **I think** she said she didn't like it because it **was 'too prickly.'**

(I think that she might actually be saying that it was literally 'thorny' and not attitude 'thorny')
Card img for reference:
![image](https://github.com/user-attachments/assets/199df07b-d5a9-4c4e-8bcb-885dd510a54e)

WisDZ reviewed (commented):
Group 7!

ヤツメ選手のファン,Yatsume Fan,(134-136)
気落ちしてる男,Downcast Man,(137-140)
もふもふ好きの女,Woman who Loves Fluffy Things,(141-142)
グッズ売りの老婆【!】,Old Woman Selling Merchandise【!】,(277+351-357),7th meeting (and last meeting)
学生の男,Male Student,(238-242),7th meeting

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+Ｂランクにまで行くと、地域の統括者になれたり、\nギルドや国の機関で重要な役職に就けたりするらしいな。\n国全体でＢランクマスターは100人もいないんじゃないか。,Once you reach B-rank; you might get assigned as a regional overseer or hold important positions in guilds or national agencies. There actually aren't that many B-rank masters. Maybe less than 100 across the whole country
+無敗のルーキー、あんたももうすぐＢランクだと思うと、\nすげー話だな。俺はいちファンとしてあんたのこと応援してるぜ。\n次の試合も楽しみだな。,It's pretty amazing to think that you might be B-rank soon. As a fan; I'm cheering for you. I'm looking forward to your next match.
+わー、ここが闘技場かー。\nお師匠さまの試合楽しみだなー。\nあ、こんにちは。お兄さんも試合を見に来たの？,Wow; so this is the colosseum. I'm really looking forward to Master Musashi's match. Oh; hello. Did you come to watch the match too; mister?
+わたしはね、ムサシ師匠の道場に通ってる門下生なの。\nお師匠様は、Cランクのマスターの中でも強いんだよ。\nそろそろ試合があるからって観戦に来たんだ。,I'm a student at Master Musashi's dojo. Master is one of the strongest out of all the C-rank masters. The match is about to start; so I hurried over.
```
<<Speaker is きょろきょろしてる女の子,Restless-Looking Girl (lines 149-151, lines 152-153 on revisit)>>

I want to include additional context in her last sentence, since it feels a bit abrupt to me atm
> I'm a student at Master Musashi's dojo. Master is one of the strongest out of all the C-rank masters. **I heard her** match is about to start; so I hurried over **to watch.**

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+強力なカードと潤滑油になるようなカード、\nそのどちらもが強いデッキを作るうえでは欠かせないと言えるでしょう。,Powerful cards and cards that fuel them... Both are essential for building a strong deck.
+「メタカード」というのをご存じでしょうか。,Do you know what a 'meta card' is?
+メタカードは特定の種類のデッキに対してよく刺さるカードのことで\n例えば魔法の「武装放棄」がそうですね。,A meta card is a card that works especially well against specific types of decks; such as the spell 'Disarmament'.
+このカードはわずか500のLPコストで、\n相手の装備カードをすべて破壊してしまいます。\n１枚で最大３枚破壊できるわけですから、きわめて強力です。,This card; at the small cost of 500 LP; can destroy all of the opponent's equipped cards. Since it can destroy up to three of your opponents cards for just one of your own; it's extremely powerful.
```
<<Speaker is 学生の男,Male Student (lines 243-248), 8th meeting>>

Second "opponents" is missing an apostrophe.
Though since it's the second mention in this line, I think we could swap out "three of your opponents cards" to "three of their cards"

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+コンボカードを活かすためには、ドローカードを追加で入れたり、\n属性やレベルでデッキからサーチするようなカードを入れると、\n動きが安定しますね。,To make the most of combo cards; try adding draw cards or cards that search your deck by attribute or level to help make your deck more consistent; okay?
+強力なカードと潤滑油になるようなカード、\nそのどちらもが強いデッキを作るうえでは欠かせないと言えるでしょう。,Powerful cards and cards that fuel them... Both are essential for building a strong deck.
+「メタカード」というのをご存じでしょうか。,Do you know what a 'meta card' is?
+メタカードは特定の種類のデッキに対してよく刺さるカードのことで\n例えば魔法の「武装放棄」がそうですね。,A meta card is a card that works especially well against specific types of decks; such as the spell 'Disarmament'.
```
<<Speaker is 学生の男,Male Student (lines 243-248), 8th meeting>>

This spell card was previously translated to "Abandon Weapons" (ID 40018)

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+このカードはわずか500のLPコストで、\n相手の装備カードをすべて破壊してしまいます。\n１枚で最大３枚破壊できるわけですから、きわめて強力です。,This card; at the small cost of 500 LP; can destroy all of the opponent's equipped cards. Since it can destroy up to three of your opponents cards for just one of your own; it's extremely powerful.
+しかし一方で、装備カードを使わない相手に対しては\nまったく役に立たないカードでもあります。,On the other hand; though; it's completely useless against opponents who don't use equipment cards.
+事前に相手のデッキの情報を集め、メタカードをデッキに入れておく。\nそういうしたたかな戦術も\nカードマスターとして勝ち上がっていくには大事なのかもしれません。,Gather information about your opponent's deck in advance and include meta cards in your own. Such shrewd tactics might be crucial for rising through the cardmaster ranks.
+おや、あなたは「武装放棄」を持っていないのですか？\nあまりレアなカードでもありませんし、\nわたしの手持ちで余っている１枚を差し上げましう。,Oh; you don't have 'Disarmament' yet? It's not that rare of a card; and I have an extra one in my collection. Would you like it?
```
<<Speaker is 学生の男,Male Student (lines 243-248), 8th meeting>>

Change "Disarmament" to "Abandon Weapons" (ID 40018)

Since the card is added without an option to accept or decline, I'd like to change the last sentence:
> Oh; you don't have 'Disarmament' yet? It's not that rare of a card; **I'll give you this extra one I have from my collection.**

Note: Unlike when the Old Merch Lady gives you a card, if you trigger this line by having 0 "Abandon Weapons", you don't get a little announcement window & it is quietly added to your available cards.
Issue #334 added. We'll also need an additional line after this one (new line 249):

> 『{%%40018}』を手に入れた！,I got {%%40018}!

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+ほら、またサインを書いておくれよ。\n代わりにまた珍しいカードをあげるからさ。,Here; write your autograph again. In return; I'll give you another rare card.
+おお、ありがとう！\nいやぁさすがは期待の新人だね。\nあんたほど優しい男をわたしゃ知らないよ。,Oh; thank you! Wow; you really earned your title of promising rookie. I don't know any other man as kind as you!
+ほらこれがお礼のカードだ。\nあんまり役に立たない猫のゴーレムだそうだよ。\nちょっと攻撃力を上げてくれるとか。まあうまく使ってやっておくれ。,Here; this is a thank-you card. It's a cat golem that doesn't really do much; but it might boost your attack power a bit. Make good use of it.
+{%%40234}を手に入れた！,You got {%%40234}!
```
Note: this was from back in Group 3, 3rd meeting with the Old Woman Selling Merchandise

All other instances in this file were translated to "I got {%%etc}"
Not sure if it matters, but I'd lean towards consistency and change "You" to "I"

HotHeadEmperor replied:
Imma tag @Giratena here and see which he thinks.

Giratena replied:
@HotHeadEmperor 

The universally accepted way to translate ITEMを手に入れた is usually "Obtained ITEM!" or "Got ITEM!". No I or you neccesary.

You'll wanna change all instances to match this!

- [ ] make all translations of _を手に入れた consistent.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+わたしは闘技場を観戦し、\n本で多くを学んでカードバトルの腕を磨いてきました。,I've been watching fights in the colosseum and studying a bunch of books hoping to hone my card battle skills.
+しかし昨今の快楽攻撃の流行のせいで、\nいくら知識があっても、女性モンスター相手に\n全く歯が立たなくなってしまったのです。,Because of the recent increase in pleasure attacks; though; no matter how much knowledge I have; I've become completely powerless against female monsters.
+無敗のルーキー、あなたの精神力を尊敬しますよ。\nわたしにもはや教えられることはありません。\nぜひその手でＢランクの栄光をつかみ取ってください！,Undefeated rookie; I respect your mental fortitude. There's nothing more I can teach you. Please; go and grasp the glory of B-rank with your own two hands!
+この国は大きく分けて4つのエリアがあります。\nカードマスターの街モルゲンがあり、穀倉地帯でもある南部。,This country is roughly divided into four areas. In the south; there's the city of Morgen; home of the cardmasters; and the granary region as well.
```
<<Speaker is 学生の男,Male Student (lines 253-257), 10th meeting>>

I believe 'farming region' or 'breadbasket' might be a localization of 'granary region' - I'm using 'agricultural' since it seems on-brand for Male Student.
Also, I want to break up the last sentence into two sentences.
> This country is roughly divided into four areas. In the south; there's the city of Morgen; home of the cardmasters. **The southern region is mostly agricultural.**

HotHeadEmperor replied:
Read a little weird; so I changed it to 'used mostly for agriculture.' Lemme know what you think.

WisDZ replied:
I like it 👍
Resolving

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+しかし昨今の快楽攻撃の流行のせいで、\nいくら知識があっても、女性モンスター相手に\n全く歯が立たなくなってしまったのです。,Because of the recent increase in pleasure attacks; though; no matter how much knowledge I have; I've become completely powerless against female monsters.
+無敗のルーキー、あなたの精神力を尊敬しますよ。\nわたしにもはや教えられることはありません。\nぜひその手でＢランクの栄光をつかみ取ってください！,Undefeated rookie; I respect your mental fortitude. There's nothing more I can teach you. Please; go and grasp the glory of B-rank with your own two hands!
+この国は大きく分けて4つのエリアがあります。\nカードマスターの街モルゲンがあり、穀倉地帯でもある南部。,This country is roughly divided into four areas. In the south; there's the city of Morgen; home of the cardmasters; and the granary region as well.
+山がちながら鉱物資源に恵まれた西部。\n海に開かれており、小さな島との交易もある東部。,The western region; which is mountainous and rich in mineral resources. The eastern region; which is open to the sea and engages in trade with small islands.
```
<<Speaker is 学生の男,Male Student (lines 253-257), 10th meeting>>

I want to strike the "; which" from both sentences:
> The western region is mountainous and rich in mineral resources. The eastern region is open to the sea and engages in trade with small islands.

WisDZ reviewed (commented):
Groups 8, 9, and 10!

ヤツメ選手のファン,Yatsume Fan,(143-144),2nd meeting
ファンの男性,Male Fan,(145-148)
きょろきょろしてる女の子,Restless-Looking Girl(149-153)
学生の男,Male Student,(243-248),8th meeting

特等席を買った女,Woman from the Front Row,(154-155)
うわさ好きの女性,Gossip-Loving Woman,(156-158)
大ファンの少年,Major Fanboy,(159-161)
ムサシの弟子の女の子,Musashi's Disciple,(162-165)
不安げな女の子,Anxious-Looking Girl,(Separate card; not reviewed)
学生の男,Male Student,(249-252),9th meeting

うれしそうにしている少年,Happy-Looking Boy,(166-169)
老人会のおばあさん【!】,Old Lady from the Senior Citizens' Club【!】,(170-175)
観客の女性,Woman from the Audience,(176-179)
❤上機嫌な女の子,❤Cheerful Girl,(Separate card; not reviewed)
学生の男,Male Student,(253-257),10th meeting

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -1,49 +1,49 @@
+❤上機嫌な女の子,❤Cheerful Girl
+低ランクのカードマスター(女),Low-Tier Female Cardmaster
+目ざとい男,Sharp-Eyed Man
+闘技場ファンのおじいさん,Old Coloseuum Fan
```
Colosseum*

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+ねーねー、ほら見て！\n無敗のルーキーさんが主役になった本がね、\nそこで売ってたの！,Hey; hey; look! There's a book with the undefeated rookie as the main character being sold over there!
+お兄さんが山にこもって修行してた話とか、\nドラゴン20匹を退治した話とかのってるんだよ。,There's a story about how he secluded himself in the mountains to train; and another about how he defeated 20 dragons!
+え？僕はそんなことしてない？\nうそだー。だって本に書いてあるもん。\nお兄さんが忘れちゃっただけだよ、きっと。,Huh? You didn't do any of that? That's not true. It's written in the book. You must have just forgotten.
+サキュバスカードかぁ。\nリュカ商会が売り出していたけど、\n使うのどうしようかしら。,A succubus card; huh? I saw the Ryuka Trading Company selling them; but I'm not sure if I should buy it or not.
```
<<Speaker is 悩んでいる女性,Troubled Woman (lines 190-192)>>

I feel like "buy **it** or not" is a mismatch from selling 'them'
Does "I'm not sure if I should buy **some** or not." fit here?

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+例えば一部のモンスターが所持している「銃」がそうですね。\n火の魔法を込めて、小さな石の弾や鉄の弾を飛ばす、\n魔法の小型杖です。,For example; some monsters carry 'guns'. These are small; magical staves that shoot small stone or iron projectiles infused with fire magic.
+彼らや彼女らは有用な道具だと主張するのですが、\n普通に魔法で火球を飛ばした方が威力もあり、筒の中で繊細な火魔術の\nコントロールがいるあの道具は、あまり有用性がわかりません。,Some claim it's a useful tool; but casting a fireball with regular magic is more powerful; and the tool requires delicate control of fire magic within the tube; so I don't know how useful it really is.
+コントロール能力さえ高ければ魔力消費を抑えられるのは\nメリットにあげられるのかもしれませんね。,If you can control the magical energy; I suppose being able to reduce magic power consumption could be considered an advantage; perhaps?
+魔族領への侵攻のための兵を王都で集めているようですね。\nサキュバス相手となると今のわたしでは\n行ったところで敵の養分になってしまうだけでしょう。,Looks like they're gathering troops in the royal capital for an invasion of the demon realm. If I were to face a succubus now; I would just end up food for the enemy.
```
<<Speaker is 学生の男,Male Student (lines 269-273, revisit lines 274-276), 13th and last meeting>>

Last sentence feels like its missing a word:

I would just end up **as** food for the enemy.
or
I would just end up **being** food for the enemy.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+しかし、わたしだってあなたの背を追うばかりではありません。\n必ずや快楽攻撃を克服して、サキュバスにも打ち勝ってみせますよ！,However; I'm not just going to let myself trail behind you. I'll definitely overcome pleasure attacks and defeat the succubi!
+わたしが託したカードは、\nデッキから魔法カードを加えることのできる汎用的なカードです。,The card I entrusted to you is a versatile card that allows you to add a spell card from your deck to your hand.
+しかし、魔術師モンスターを強化する効果もついているので、\n魔術師カードと共に使えば更なる効果を発揮できるでしょう。,However; it also has the effect of enhancing magician monsters; so if you use it with magician cards; it'll be even stronger.
+もし使い道がなければ売ってしまってもかまいません。\nそのときはあなたが魔族領に行くときの、路銀にでも変えてください！,If can't really use it; feel free to sell it. If you did that; you could exchange it for travel expenses when you go to the demon realm!
```
<<Speaker is 学生の男,Male Student (lines 269-273, revisit lines 274-276), 13th and last meeting>>

If **you** can't really use it

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+オクタ,Octa
+ミレーヌ,Mireille
+ドワン,Dwan
+パトラ,Patra
```
I checked the code, and lines 362-369 appear to be for a cut character named "Colosseum Fanatic (闘技場マニア)"
OK to delete.

HotHeadEmperor replied:
Imma mark this as a @Giratena decision

Giratena replied:
Lets keep it in. Removing it will make changes harder to see.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+パトラ,Patra
+あ、あの・・・。あなたがあの{主人公}選手ですか？\nうわぁー、ほんものだ。,Uh; um... Are you {主人公}? Wow; it's really you.
+お母さんから、エッチなのはダメって言われるから、\n試合は見れてないんですけど、\n話ですごく勝ってる男のマスターがいるって聞いてて・・・。,My mom told me that I'm not allowed to do anything inappropriate; so I haven't been able to watch any matches; but I've heard that there's a male master who's winning a lot...
+僕もカードマスターになりたいんです！\n握手してください！{主人公}選手みたいになれるように頑張ります！,I want to become a cardmaster; too! Please shake my hand! I'll do my best to become just like you!
```
Similarly, lines 370-372 also appear to be cut - these ones for an NPC named "Boy NPC (少年NPC)"
OK to delete.

HotHeadEmperor replied:
@Giratena

Giratena replied:
Also keep in ^^

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+とはいえ、「デッキケース」の魔道具は\nある程度の枚数のカードを入れないと効果を発揮しません。\n最低20枚は入れないといけないので、１枚デッキは組めないわけです,That said; the "deck case" magical tool doesn't work unless you put a certain number of cards in it. You need at least 20 cards; so you can't make a one-card deck.
+「デッキケース」に入れて事前に魔力を充填しておかないと\nカードは連発して使えないので、\nいたしかたないところではありますね。,If you don't put the cards in the "deck case" and charge them with magical power in advance; you won't be able to use them repeatedly; so it's pretty unavoidable; huh?
+デッキケースは遺跡からよく発掘され、広く使われていますが、\n謎の多い魔道具です。入れたカードが自動でシャッフルされるのも、\n魔力の充填が効率よくなるからだと言われています。,Deck cases are often excavated from ruins and are widely used; but they're still a pretty mysterious magical tool. It's said that the cards placed inside are automatically shuffled because it makes charging them with magical power more efficient.
+強いカード10枚ぐらいでデッキを組めて、\nさらに事前に山札を並べておければ最高なのですが…。\nそううまい話はないようです。,If you could build a deck with about ten strong cards and maybe even arrange the draw pile in advance; you might be unstoppable... But it's just not that easy I guess.
```
Lines 218-222 do not currently appear in any dialogue due to a bug, opening an Issue.

Speaker would be (rather, it will be) Male Student, meeting 3

WisDZ replied:
Linking issue #335

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+カードパックを開いたとき\n人によってどんなカードが顕現するか違うため、\nこのようなロマンチック？な学説も出ているわけですね。,When you open a card pack; the cards that manifest differ from person to person; which is why such a romantic theory has emerged; I guess?
+わたしとしては一種の契約魔術のようなものだと思っていますが、\nその仕組みが解明されるようになるのは、\nもっとずっと先のことになりそうです。,I think of it as a kind of contract magic; but it seems like it'll be a long time before the mechanisms behind it are fully understood.
+ちなみに魔法カードやトラップカードは、\n色々なマスターがパックを開けたときに同じカードが出ることがあります。,By the way; magic cards and trap cards can sometimes appear as the same card when different masters open their packs.
+このように理性を持たないカードは人を選ばず出ることから、\n先ほど紹介したような、理性のあるカードは前世のつながりがあるのでは\nという突飛な説が生まれたわけです。,It's the fact that non-sentient cards appear no matter who opens a pack that has led to the outlandish theory I mentioned earlier that sentient cards might have connections to past lives.
```
Lines 263 and 264 do not currently appear in any dialogue due to a bug, opening an Issue.

Speaker will be Male Student, meeting 11

WisDZ replied:
Linking issue #335

WisDZ reviewed (commented):
Groups 11, 12, and 13, and added comments on any text that is currently unused.
Review should now be complete!

低ランクのカードマスター(女),Low-Tier Female Cardmaster,(180-182)
目ざとい男,Sharp-Eyed Man,(183-184)
闘技場ファンのおじいさん,Old Colosseum Fan,(185-186)
❤ジェミニ,❤Gemini,(Separate card; not reviewed)
学生の男,Male Student,(258-262),11th meeting

本を持っている女の子,Girl Holding a Book,(187-189)
悩んでいる女性,Troubled Woman,(190-192)
恐れているおじいさん,Fearful Old Man,(193-195)
学生の男,Male Student,(265-268),12th meeting

わくわくしてる少年,Excited Boy,(196-197)
調子の悪い女性,Sick-Feeling Woman,(198-200)
非番の兵士,Off-Duty Soldier,(201-205)
学生の男,Male Student,(269-276),13th meeting* and final meeting

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+男のカードマスターはあんまり勝率がよくないんだよねぇ。,Male cardmasters don't have a very good win-rate; you know.
+昔はそうでもなかったんだけど、サキュバスが出現してからは男の強いカードマスターがたくさん魔族領でやられちまって、それで女マスターと実力差がついちまった。,It wasn't always like that; but ever since the succubi appeared; many strong male cardmasters have been defeated in the demon realm; and now the skill gap with the female masters has just gotten wider.
+そのうえ最近はサキュバス系のカードを使うマスターも増えて、男はすぐこてんぱんに負けちまう。,What's worse; recently there have been more masters using succubus-type cards; and men just lose to them way too quickly.
+祭りが始まる前まではサキュバスのカードなんて見たことがなかったんだけどなぁ。時代は変わったもんだ。,Before the festival started; I'd never even seen a succubus card. Times have really changed.
```
Lines 69-74 appear to have been cut from Male Colosseum Fan's first meeting.

Event is ended with:
    イベント終了
and what seems like intentional spacing, so I believe these lines are OK to delete.

HotHeadEmperor replied:
@Giratena

Giratena replied:
Let's leave these in for now - At worst we could add a comment saying they aren't used.

WisDZ commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+悪魔タイプ自体かなり珍しかったと思うし、\nなんでこんなにサキュバスモンスターが増えたのかしら。\n新しい遺跡で珍しいパックでも出たとか？,I think demon-types themselves were pretty rare before; so why have there been so many succubus monsters recently? Did they find a rare pack in those new ruins or something?
+ねえ、ねえ、男のひとって、\nなんであんなに簡単に女のモンスターに負けちゃうの？,Hey; hey; why do guys lose so easily to female monsters?
+観客席からだと、遠くて何やってるかよくわからないの。\nモンスターもマスターも魔力のバリアがあって、\n剣で斬っても簡単にはやられないはずなのに・・・。,The audience's seats are too far away to see exactly what's happening. Both the monster and the master have magical barriers; so they shouldn't be beaten easily; even if they get slashed or stabbed...
+女の子モンスターに抱き着かれたら、\nすぐビクビクってふるえてそのままやられちゃうんだよ。\nへんなのー。,But if a girl monster hugs him; the guy starts starts trembling and get taken down; just like that. It's really weird.
```
Lines 78-80 are credited to 女の子 in the first round, though such a character did not appear for me. 

Unlike other unused lines, the code for this character makes me believe they were intended to exist. Since the NPC would appear where Eruru does in my test file, I imagine that there's either conditions for the appearance or potentially a bug. I'll open an issue once I've done more testing. 

The word "starts" is doubled on line 80

WisDZ replied:
Update: I'm fairly sure this character was dropped in favor of Eruru's dialogue.

In reviewing the file, it looks to me like there is no way to get these lines to appear.

WisDZ reviewed (commented):
Sorry - missed two chunks. Now Review is complete, but for real this time 😁

HotHeadEmperor committed:
Update Colosseum.txt

WisDZ committed:
File Reorganization

Mostly chronological order

Male Student and Old Woman Selling Merchandise are separated.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -1,49 +1,5 @@
 default
-闘技場ファンの男,
-観客の女,
```
Huh, why were all of these deleted? Was it an accident?

WisDZ replied:
Rows were moved, but change now reverted 👍

WisDZ committed:
Revert "File Reorganization"

This reverts commit 84b054a58e9c77afa4576b9168238009c1f9eb66.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+初めまして。闘技場への参加をご希望でしょうか？,Nice to meet you. Would you like to participate in the colosseum?
+分かりました。それではギルドリングをお見せください。,Understood. Then please; show me your guild ring.
+・・・。,...
+ありがとうございます。貴方がDランクであることを確認しました。闘技場への参加を承認します。,Thank you very much. After confirming your rank is D; I gladly approve your participation in the colosseum.
```
This sentence feels a little stiff. Maybe give it another go.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+街中とはまるで違う雰囲気を肌で感じる。周りに見えるカードマスター達の闘志だろうか。,Perhaps it was the fighting spirit of the cardmasters around me; but the atmosphere felt completely different from the city. 
+ここでのカードバトルは更に厳しいものになるだろう。この空気に飲まれないよう、僕は心を奮い立たせた。,The card battles here will no doubt be more intense than the ones I've faced so far. I steeled my resolve; determined to not be overwhelmed by the atmosphere.
+初めまして。闘技場への参加をご希望でしょうか？,Nice to meet you. Would you like to participate in the colosseum?
+分かりました。それではギルドリングをお見せください。,Understood. Then please; show me your guild ring.
```
This sentence feels a little stiff. Something like, "Now then; please show me your guild ring." Would flow a lot better.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+分かりました。それではギルドリングをお見せください。,Understood. Then please; show me your guild ring.
+・・・。,...
+ありがとうございます。貴方がDランクであることを確認しました。闘技場への参加を承認します。,Thank you very much. After confirming your rank is D; I gladly approve your participation in the colosseum.
+再度話しかけていただければ、本日からでも対戦を組むことができますので、準備ができたらお声がけくださって構いませんよ。,If you speak to me again; we can arrange a match starting today; so feel free to let me know when you're ready.
```
This sentence is a bit stiff as well. Give it another go ^^;

HotHeadEmperor replied:
Gave it a go.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+再度話しかけていただければ、本日からでも対戦を組むことができますので、準備ができたらお声がけくださって構いませんよ。,If you speak to me again; we can arrange a match starting today; so feel free to let me know when you're ready.
+おや、あんた新人選手かい？\nそのリングの色、Dランクだね。,Oh hey; you're a rookie; aren't you? That ring color means you're a D-rank; right?
+闘技場では普段からカードマスターたちが試合をして\nその腕を競っているんだ。\n祭りの間はとくに試合が盛りだくさんさ。,In the colosseum; cardmasters usually compete in matches to showcase their skills. During the festival; matches here are especially common.
+国中から集まってきたカードマスターたちの火花散らす戦い…。\nいやぁ、たまらないね。,Cardmasters from all over the country gather and have fierce battles... Ah; the excitment of it all is just intoxicating; don't you think?
```
This feels a little stiff. Maybe try the following!
gather and have -> gathering to have
Ah; the excitment of it all is just intoxicating; don't you think? -> Oh; the excitment of it all... I can hardly contain myself!

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+闘技場では普段からカードマスターたちが試合をして\nその腕を競っているんだ。\n祭りの間はとくに試合が盛りだくさんさ。,In the colosseum; cardmasters usually compete in matches to showcase their skills. During the festival; matches here are especially common.
+国中から集まってきたカードマスターたちの火花散らす戦い…。\nいやぁ、たまらないね。,Cardmasters from all over the country gather and have fierce battles... Ah; the excitment of it all is just intoxicating; don't you think?
+でもよ、最近は女マスターばっかり勝っててつまらねえ。\nもっと剣と剣がぶつかるような戦いを、前みたいに見たいもんだ。\n男としてあんたには期待してるぜ！,But you know; lately it's only been female masters winning; and it's actually starting to get pretty boring. I want to see battles where swords clash like they used to. So as a guy; I'm counting on you!
+闘技場って、中にいるだけで何だかワクワクしてくるよな。,The colosseum... Just being here makes you a bit excited; doesn't it?
```
Hmm... I think "excited" might be a little poor choice of words here for the context xD

HotHeadEmperor replied:
Tried something else.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+おや、あんた新人選手かい？\nそのリングの色、Dランクだね。,Oh hey; you're a rookie; aren't you? That ring color means you're a D-rank; right?
+闘技場では普段からカードマスターたちが試合をして\nその腕を競っているんだ。\n祭りの間はとくに試合が盛りだくさんさ。,In the colosseum; cardmasters usually compete in matches to showcase their skills. During the festival; matches here are especially common.
+国中から集まってきたカードマスターたちの火花散らす戦い…。\nいやぁ、たまらないね。,Cardmasters from all over the country gather and have fierce battles... Ah; the excitment of it all is just intoxicating; don't you think?
+でもよ、最近は女マスターばっかり勝っててつまらねえ。\nもっと剣と剣がぶつかるような戦いを、前みたいに見たいもんだ。\n男としてあんたには期待してるぜ！,But you know; lately it's only been female masters winning; and it's actually starting to get pretty boring. I want to see battles where swords clash like they used to. So as a guy; I'm counting on you!
```
Hmm "As a guy" feels a little forced I think...

HotHeadEmperor replied:
I think it's important, though, so I changed it some.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+悪魔タイプ自体かなり珍しかったと思うし、\nなんでこんなにサキュバスモンスターが増えたのかしら。\n新しい遺跡で珍しいパックでも出たとか？,I think demon-types themselves were pretty rare before; so why have there been so many succubus monsters recently? Did they find a rare pack in those new ruins or something?
+ねえ、ねえ、男のひとって、\nなんであんなに簡単に女のモンスターに負けちゃうの？,Hey; hey; why do guys lose so easily to female monsters?
+観客席からだと、遠くて何やってるかよくわからないの。\nモンスターもマスターも魔力のバリアがあって、\n剣で斬っても簡単にはやられないはずなのに・・・。,The audience's seats are too far away to see exactly what's happening. Both the monster and the master have magical barriers; so they shouldn't be beaten easily; even if they get slashed or stabbed...
+女の子モンスターに抱き着かれたら、\nすぐビクビクってふるえてそのままやられちゃうんだよ。\nへんなのー。,But if a girl monster hugs him; the guy starts trembling and get taken down; just like that. It's really weird.
```
And get taken down -> and gets taken down

Either that or reword

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+観客席からだと、遠くて何やってるかよくわからないの。\nモンスターもマスターも魔力のバリアがあって、\n剣で斬っても簡単にはやられないはずなのに・・・。,The audience's seats are too far away to see exactly what's happening. Both the monster and the master have magical barriers; so they shouldn't be beaten easily; even if they get slashed or stabbed...
+女の子モンスターに抱き着かれたら、\nすぐビクビクってふるえてそのままやられちゃうんだよ。\nへんなのー。,But if a girl monster hugs him; the guy starts trembling and get taken down; just like that. It's really weird.
+おお、おお、あんたは例のルーキーじゃないか。\nオクタ選手との試合見たよ！\n女マスターに勝つなんてすごいじゃないか。,Oh; oh; you're that rookie; aren't you? I saw your match against Octa! Winning against a female master like that is pretty impressive; don't you think?
+そのうえクエストでサキュバスも倒したんだろ？\nいやぁ、ほんと期待の新人だな！\nアンタの試合、次も見に行くぜ！がんばってくれよ！,And on top of that; you defeated a succubus on a quest; didn't you? Wow; you're really an up-and-coming rookie! I'll definitely be watching your next match; too! Keep it up!
```
And on top of that -> not only that
You defeated a Succubus on a quest -> you also defeated a succubus on that quest; didn't you?

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+女の子モンスターに抱き着かれたら、\nすぐビクビクってふるえてそのままやられちゃうんだよ。\nへんなのー。,But if a girl monster hugs him; the guy starts trembling and get taken down; just like that. It's really weird.
+おお、おお、あんたは例のルーキーじゃないか。\nオクタ選手との試合見たよ！\n女マスターに勝つなんてすごいじゃないか。,Oh; oh; you're that rookie; aren't you? I saw your match against Octa! Winning against a female master like that is pretty impressive; don't you think?
+そのうえクエストでサキュバスも倒したんだろ？\nいやぁ、ほんと期待の新人だな！\nアンタの試合、次も見に行くぜ！がんばってくれよ！,And on top of that; you defeated a succubus on a quest; didn't you? Wow; you're really an up-and-coming rookie! I'll definitely be watching your next match; too! Keep it up!
+闘技場では、勝ったマスターが負けたマスターを\n自由にもてあそぶ権利があるわ。\n表向きは魔力補給のためってことになってるけれど・・・。,In the colosseum; the winning master has the right to toy with the losing master freely. Officially; it's for the purpose of replenishing magical power; but...
```
This first sentence is quite Engrishy. Maybe try:
"In the colosseum; the winning cardmaster has the right to toy with the loser in whatever way they like."

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+闘技場では、勝ったマスターが負けたマスターを\n自由にもてあそぶ権利があるわ。\n表向きは魔力補給のためってことになってるけれど・・・。,In the colosseum; the winning master has the right to toy with the losing master freely. Officially; it's for the purpose of replenishing magical power; but...
+いまよりずっと昔、闘技場が罪人の戦いを見る場所だったころ、\nそのころは罪人をなぶるのを見世物にして、\n勝者は敗者を好きにできる決まりになっていたそうなの。,Long ago; the colosseum was a place to watch criminals fight one another. It's said it was a spectacle to torment the criminals; and the victor had the right to do as they pleased with the vanquished.
+野蛮な風習がなくなった今でも、\nそのころの名残で闘技場の勝者が敗者を好きにできる習慣が\n残っているのだそうよ。,Even though the more barbaric customs have disappeared; it seems that remnants of that time still linger; allowing the winners of the colosseum to do as they please with the losers.
+ちなみに男のマスター同士の場合、魔力補給の名分がないから\n勝っても何かしたりできないの。\nそこは野蛮じゃなくていいわね。,By the way; when it's two male masters fighting; there's no pretext for mana replenishment; so even if you win; you can't really do anything to them. At least that part isn't so barbaric; right?
```
ちなみに -> Speaking of which;
This sentence feels rather rough. Maybe something like this?
"Speaking of which; in the case of two male masters fighting; there's no obligation or way to refill your mana afterwards; so even if you win; it's not like you can do anything. At least that part isn't so barbaric; right?"

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+野蛮な風習がなくなった今でも、\nそのころの名残で闘技場の勝者が敗者を好きにできる習慣が\n残っているのだそうよ。,Even though the more barbaric customs have disappeared; it seems that remnants of that time still linger; allowing the winners of the colosseum to do as they please with the losers.
+ちなみに男のマスター同士の場合、魔力補給の名分がないから\n勝っても何かしたりできないの。\nそこは野蛮じゃなくていいわね。,By the way; when it's two male masters fighting; there's no pretext for mana replenishment; so even if you win; you can't really do anything to them. At least that part isn't so barbaric; right?
+なあ、アンタ、Dランクで最強のマスターって誰だと思う？\nやっぱり俺はミレーヌさんだね。,Hey; who do you think is the strongest master in D-rank? If you ask me; it's gotta be Mylene.
+いや、ミレーヌさんがレベルの高いモンスターを使えないのは\n俺も知ってるよ。でも、あのおっぱい。あんなの見たら、\n俺だったら即戦意を喪失しちゃうよ。,I know she can't use high-level monsters; but those boobs... If I saw those; I'd lose all my fighting spirit right away!
```
If I saw those; I'd lose all my fighting spirit right away! -> If I saw those while battling; my fighting spirit would vanish before I knew it!

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+ちなみに男のマスター同士の場合、魔力補給の名分がないから\n勝っても何かしたりできないの。\nそこは野蛮じゃなくていいわね。,By the way; when it's two male masters fighting; there's no pretext for mana replenishment; so even if you win; you can't really do anything to them. At least that part isn't so barbaric; right?
+なあ、アンタ、Dランクで最強のマスターって誰だと思う？\nやっぱり俺はミレーヌさんだね。,Hey; who do you think is the strongest master in D-rank? If you ask me; it's gotta be Mylene.
+いや、ミレーヌさんがレベルの高いモンスターを使えないのは\n俺も知ってるよ。でも、あのおっぱい。あんなの見たら、\n俺だったら即戦意を喪失しちゃうよ。,I know she can't use high-level monsters; but those boobs... If I saw those; I'd lose all my fighting spirit right away!
+実際、男相手の戦績はかなりいいみたいだしな。\nアンタも男のマスターだし、きっとミレーヌさんには勝てねえぜ。,Actually; it seems her track record against men is pretty impressive. You're a male master too; so there's no way you'd win against Mylene.
```
Actually; it seems her track record against men is pretty impressive. -> Actually; her track record against men **seems pretty impressive**.
You're a male master too; -> **And** you're a male master as well;

HotHeadEmperor replied:
Trying 'and since' because just 'and' didn't make sense to me.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+実際、男相手の戦績はかなりいいみたいだしな。\nアンタも男のマスターだし、きっとミレーヌさんには勝てねえぜ。,Actually; it seems her track record against men is pretty impressive. You're a male master too; so there's no way you'd win against Mylene.
+ん・・・。あなたのこと見たことあるような・・・。\nああっ、噂のルーキーくんじゃない？\nきゃー、近くで見るとけっこうかわいいのね❤,Hmm... I feel like I've seen you somewhere before... Oh; aren't you the rookie everyone's talking about? Wow; you're actually pretty cute up close; huh?❤
+ミレーヌさんとの試合見てたわよー。\nあんなセクシーな人にまで勝っちゃうなんて、\n我慢強くてすごいじゃない。,I watched your match with Mylene. You must have pretty good endurance to win against someone as sexy as her. You're really impressive.
+サキュバスにも勝ったんでしょ？\n男ってすぐ色気でふらふらになっちゃうやつらばっかりだから、\nあなたみたいな硬派な子ってかっこいいわ。,You even beat a succubus too; right? Most guys are so easily swayed by charm; so a tough guy like you is really cool.
```
Careful of the flow here:
So a tough guy like you is really cool. -> So tough guys like you are especially cool.

HotHeadEmperor replied:
I took 'especially' but making it plural didn't make sense to me.

Giratena replied:
~~I think the nuance makes more sense as plural. If its singular it may imply that our character is the only tough man (which might be true haha) but it's an unusual nuance English wise.~~ nvm yeah I think singular is ok haha

I also think "charm" is weird though. Maybe "feminine charm"?

HotHeadEmperor replied:
went with 'feminine charms.' see how that looks.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+ミレーヌさんとの試合見てたわよー。\nあんなセクシーな人にまで勝っちゃうなんて、\n我慢強くてすごいじゃない。,I watched your match with Mylene. You must have pretty good endurance to win against someone as sexy as her. You're really impressive.
+サキュバスにも勝ったんでしょ？\n男ってすぐ色気でふらふらになっちゃうやつらばっかりだから、\nあなたみたいな硬派な子ってかっこいいわ。,You even beat a succubus too; right? Most guys are so easily swayed by charm; so a tough guy like you is really cool.
+わたしファンになっちゃったから、応援してるわよ。\n頑張ってねー。,I've become a fan; so I'm cheering for you. Keep it up; okay?
+こんにちはー、噂のルーキーくん。\nあは、手を振り返してくれた。うれしいー❤,Hey; it's the rumored rookie! Ah; he waved back. I'm so happy!❤
```
Ah; -> Aha!
あは (aha) is more like a laugh

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+わたしファンになっちゃったから、応援してるわよ。\n頑張ってねー。,I've become a fan; so I'm cheering for you. Keep it up; okay?
+こんにちはー、噂のルーキーくん。\nあは、手を振り返してくれた。うれしいー❤,Hey; it's the rumored rookie! Ah; he waved back. I'm so happy!❤
+ミレーヌさん以外にも強いマスターいっぱいいるけど、\nちゃんとカッコいいところ見せてよね。応援してるわよ！,But there are plenty of strong masters besides Mylene; so make sure to show us just how cool you can be. I'm rooting for you!
+モルゲンのカードマスター学園には有名なトップ選手がいてな、\n人魚使いのメロディ、雷電のボルタ、無敵のシルヴィアって三人が\n全員Bランクマスターなんだぜ。,At Morgen's Cardmaster Academy; there are some pretty famous top players: Melody the mermaid user; Volta the thunder user; and Sylvia the invincible; all of whom are B-rank masters.
```
Sylvia the invincible; all of whom -> Sylvia the invincible. All of whom

HotHeadEmperor replied:
I just em dashed it, because that last sentence relies on the first sentence to flow correctly. If you don't like that, I'll change it to 
At Morgen's Cardmaster Academy; there are some pretty famous top players; all of whom are B-rank masters. Melody the mermaid user; Volta the thunder user; and Sylvia the invincible.

Giratena replied:
Em dash is good. It's quite handy in Japanese translations ^^

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+モルゲンのカードマスター学園には有名なトップ選手がいてな、\n人魚使いのメロディ、雷電のボルタ、無敵のシルヴィアって三人が\n全員Bランクマスターなんだぜ。,At Morgen's Cardmaster Academy; there are some pretty famous top players: Melody the mermaid user; Volta the thunder user; and Sylvia the invincible; all of whom are B-rank masters.
+Bランクまで上がった学生は国中でこの三人だけだからな。\nまさにエリート様って感じだよ。\nただボルタ以外はみんな偉ぶってなくて、庶民からの人気も高いんだ。,They're actually the only students who have reached B-rank in the entire country. They truly feel like elite. However; except for Volta; they're all pretty humble; making them pretty popular among the common folk.
+このさき守護者になったり王都の騎士団長になったりするんだろうけど\nああいうしっかりした子たちが上に立ってくれるなら、\n頼もしい限りだね。,In the future; they might become guardians; or the captain of the royal capital's knights! If such reliable kids take the lead; it would be incredibly reassuring; don't you think?
+ん、アンタはミレーヌさんを倒した男マスターじゃねえか。,Hmm; you're the guy who defeated Mylene; aren't you?
```
ん、-> Mm.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+Bランクまで上がった学生は国中でこの三人だけだからな。\nまさにエリート様って感じだよ。\nただボルタ以外はみんな偉ぶってなくて、庶民からの人気も高いんだ。,They're actually the only students who have reached B-rank in the entire country. They truly feel like elite. However; except for Volta; they're all pretty humble; making them pretty popular among the common folk.
+このさき守護者になったり王都の騎士団長になったりするんだろうけど\nああいうしっかりした子たちが上に立ってくれるなら、\n頼もしい限りだね。,In the future; they might become guardians; or the captain of the royal capital's knights! If such reliable kids take the lead; it would be incredibly reassuring; don't you think?
+ん、アンタはミレーヌさんを倒した男マスターじゃねえか。,Hmm; you're the guy who defeated Mylene; aren't you?
+あんたが勝っちまうせいで「妖精の酒場」の割引がなくなって、\n昨日は酒代が高くついちまった。,Since you won; she didn't offer a discount at the "Fairy Tavern"; and the drinks ended up costing more yesterday.
```
Remember the topic:
She didn't offer **us** a discount

HotHeadEmperor replied:
Don't know that it's necessary since he's referring to the more general act of offering it than to him specifically, but I'll add it. (She's the topic, and offering is the action, kinda like 'he didn't sell his wares today' vs 'he didn't sell us his wares today.')

Giratena replied:
I would say thats exactly why it needs to be "us". "A discount" implies she's exclusively giving it to only him, where as us implies its general.

In your example it's more "he didn't sell me his wares" vs "he didn't sell us his wares"

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+お、期待のルーキーじゃないか。\n俺、あんたのこの間の試合見たぜ。,Oh; aren't you that promising rookie? I watched your match the other day.
+同じ日に上位ランカーの試合があったから、\n実況もつかなかったみたいだけど、\nやっぱりあんた強いな。これで３連勝じゃないか。,Since there was a match with a top-ranked player on the same day; it seems there was no commentary for your fight; but you're really strong. That's three consecutive wins now; right?
+昔は男のマスターもAランクやBランクにいたんだけど、\nみんなここ数年でランクダウンしちまった。\nあんたは頑張って上のランクに行ってくれよ。楽しみにしてるぜ。,In the past; there were male masters in the A- and B-ranks; but all the guys have really dropped off over the past few years. Keep working hard to reach the higher ranks. I'm looking forward to it!
+最近上位ランカーは女の選手ばっかりになったわ。\nだってみんな女モンスターに誘惑されたら\nすぐ負けちゃうんですもの。,These days; the top-ranked players are all women. It's because guys just crumble as soon as they get tempted by female monsters.
```
They get -> they're

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+同じ日に上位ランカーの試合があったから、\n実況もつかなかったみたいだけど、\nやっぱりあんた強いな。これで３連勝じゃないか。,Since there was a match with a top-ranked player on the same day; it seems there was no commentary for your fight; but you're really strong. That's three consecutive wins now; right?
+昔は男のマスターもAランクやBランクにいたんだけど、\nみんなここ数年でランクダウンしちまった。\nあんたは頑張って上のランクに行ってくれよ。楽しみにしてるぜ。,In the past; there were male masters in the A- and B-ranks; but all the guys have really dropped off over the past few years. Keep working hard to reach the higher ranks. I'm looking forward to it!
+最近上位ランカーは女の選手ばっかりになったわ。\nだってみんな女モンスターに誘惑されたら\nすぐ負けちゃうんですもの。,These days; the top-ranked players are all women. It's because guys just crumble as soon as they get tempted by female monsters.
+男のマスターってたいしたことないのね。\nあなたもマスターみたいだけど、\nふふ、わたしでもエッチなことしてあげれば倒せそう❤,The male masters aren't that impressive; huh? You seem like a master; too; but... Fufu; I bet I could take you down just by doing something naughty~❤
```
Master -> cardmaster
Just by doing something naughty~ -> with a little sexy tease~

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+よお、期待のルーキー！\nここまで４連勝なんだろ？アンタすげーな！,Hey; it's the promising rookie! Four consecutive wins so far; right? You're amazing!
+油断すれば下位ランカーにだって\n足元をすくわれることもあるのがカードバトルの世界。\nそれを負け知らずで４回も勝つなんてたいしたもんだぜ。,If they let your guard down; even lower-ranked players can trip you up in the world of card battles. Winning four times in a row is quite the achievement.
+あの蛇使いのパトラにも勝つなんてよ。\n次は昇格戦かぁ、見るのが楽しみだなぁ！\n最近は男は降格してばっかりだったからアンタには期待してるぜ！,You even beat that snake charmer; Patra! Next is the promotion match; right? I'm looking forward to watching it! Recently; the guys have been getting demoted left and right; so I'm counting on you!
+だからほらよ、持ってきな！\nこの金で新しいパックでも買って、\n上のランクのやつらをぎゃふんと言わせてやりな！,So here; take this! Use that to buy a new pack and make those higher-ranked girls eat their words!
```
Money wasn't translated in this sentence.

Use that -> use this money

HotHeadEmperor replied:
I think I cut it because it made the line feel stilted, but I'll add it back.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+おっと、すまない。ぶつかるところだったね。\nきみもカードマスターかい？\n僕もそうさ。元はＣランクマスターだったんだぜ。,Oops; sorry. I almost bumped into you there. Are you a cardmaster? Yeah; me too. I used to be C-rank; actually.
+２週間前ぐらいになるんだが、\nCランクのキャミーという選手と戦ってね、\nその時の戦闘がどうにも忘れられないんだ。,It was about two weeks ago that I fought a C-rank player named Cammy; and I just can't get that battle out of my head.
+あの選手に負けたせいで\nCランクからDランクに落ちてしまったっていうのに、\n俺も自分で何を言ってるんだって思うよ。だけど・・・。,I dropped from C-rank to D-rank because of her; and I can't believe I'm saying this... But...
+キャミー選手の胸は\nほんとに脳がとけてしまうんじゃないかってぐらい気持ちよくて、\nそのうえ快楽攻撃ばっかり執拗にしてくるから…❤ううっ…❤,Cammy's breasts felt so good; it's like my brain was melting. And she just kept relentlessly hitting me with those pleasure attacks…❤ Ughh…❤
```
The flow here is a little off. I'd expect something more like:

Cammy's breasts felt so good; it was like I could actually feel my mind melting away... And when she kept relentlessly hitting me with those pleasure attacks...❤ Oogh...❤

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+油断すれば下位ランカーにだって\n足元をすくわれることもあるのがカードバトルの世界。\nそれを負け知らずで４回も勝つなんてたいしたもんだぜ。,If they let your guard down; even lower-ranked players can trip you up in the world of card battles. Winning four times in a row is quite the achievement.
+あの蛇使いのパトラにも勝つなんてよ。\n次は昇格戦かぁ、見るのが楽しみだなぁ！\n最近は男は降格してばっかりだったからアンタには期待してるぜ！,You even beat that snake charmer; Patra! Next is the promotion match; right? I'm looking forward to watching it! Recently; the guys have been getting demoted left and right; so I'm counting on you!
+だからほらよ、持ってきな！\nこの金で新しいパックでも買って、\n上のランクのやつらをぎゃふんと言わせてやりな！,So here; take this! Use that to buy a new pack and make those higher-ranked girls eat their words!
+アンタのすげー試合を見せてもらえるなら、\nこれぐらい安いもんだぜ。\n楽しみにしてるぜ、ルーキー！,It's a small price to pay to get to see your amazing matches. I'm looking forward to it; rookie!
```
To get to see -> to see

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+ルーキーってこの街に来たときはFランクだったのに、\n連戦連勝であっという間にCランクに上がったんだろ？\nすげー！かっこいい！俺も無敗のルーキーみたいになりたい！,You were just an F-rank when you first came to town; but you rose to C-rank so quickly just winning back-to-back; right? That's amazing! So cool! I want to be an undefeated rookie just like you!
+なあ、握手してくれよ。いいの？ほんとに？\nやった！ありがとう！\n俺、この手しばらく洗わないよ！,Hey; can I shake your hand? Is that okay? Really!? Awesome! Thank you! I won't be washing this hand for a while!
+最近どうも王都の方がきなくさいらしいな。\nギルドマスターや高ランクのマスターが盛んに行き来していて、\nそろそろ大きな戦でもあるんじゃないかって話だぜ。,Recently; it feels like something's brewing in the royal capital. Guild masters and high-ranking cardmasters have been coming and going frequently; and there's talk that a major conflict might be on the horizon.
+王都とモルゲンには強力な結界があって、\nサキュバスたちも簡単には手を出せねえ。\nそれでしばらく膠着状態だったんだが、いよいよ動き出すのかもな。,The royal capital and Morgen have powerful barriers; so even the succubi can't easily make a move. That's why the war's been in a stalemate for a while; but it might finally be starting to move again.
```
That last sentence feels weird and vague. Maybe take another look at it!

HotHeadEmperor replied:
Attempted something

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+わたしもカードマスターなんだけれど、\nとてもDランク以上に上がれる気がしないわ。\nバトルで勝つのって大変よ。,I'm a cardmaster; too; but I really don't feel like I can rise above D-rank. Winning battles these days is tough.
+Dランクには落ちぶれた男性マスターがたくさんいるし、\nわたしも色気を磨けば上に上がれるのかしら。\nうーん・・・。,There are a bunch of male masters in the D-rank who fell from the higher ranks; and sometimes I wonder if I could rise up by honing my allure like the other girls. Hmm...
+一度サキュバスカードを探してデッキに入れてみようかな。\nなかなか見つからないって言うけれど、市場で見つかるかしら。,Maybe I should try looking for a succubus card and add it to my deck. They say they're hard to find; but maybe I could find one in the market...
+カードマスターが犯罪をしたらどうなると思う？\nギルドで指名手配されて、\nモルゲンの特殊牢獄に入れられるのよ。,What do you think happens when a cardmaster commits a crime? They end up wanted by the guild and locked up in Morgen's special prison.
```
They end up wanted -> They end up on the guild's wanted list, and are locked up

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+Dランクには落ちぶれた男性マスターがたくさんいるし、\nわたしも色気を磨けば上に上がれるのかしら。\nうーん・・・。,There are a bunch of male masters in the D-rank who fell from the higher ranks; and sometimes I wonder if I could rise up by honing my allure like the other girls. Hmm...
+一度サキュバスカードを探してデッキに入れてみようかな。\nなかなか見つからないって言うけれど、市場で見つかるかしら。,Maybe I should try looking for a succubus card and add it to my deck. They say they're hard to find; but maybe I could find one in the market...
+カードマスターが犯罪をしたらどうなると思う？\nギルドで指名手配されて、\nモルゲンの特殊牢獄に入れられるのよ。,What do you think happens when a cardmaster commits a crime? They end up wanted by the guild and locked up in Morgen's special prison.
+カードを取り上げたうえで、\n強いカードマスターの看守たちが24時間見張っているの。\n凶悪な犯罪者だって逃げだすことはできないわ。,They take away your cards and strong cardmaster guards watch over you 24/7. Even the most notorious criminals can't escape.
```
This sentence doesn't quite flow right. Maybe take another look at it!

HotHeadEmperor replied:
Tried something

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+一度サキュバスカードを探してデッキに入れてみようかな。\nなかなか見つからないって言うけれど、市場で見つかるかしら。,Maybe I should try looking for a succubus card and add it to my deck. They say they're hard to find; but maybe I could find one in the market...
+カードマスターが犯罪をしたらどうなると思う？\nギルドで指名手配されて、\nモルゲンの特殊牢獄に入れられるのよ。,What do you think happens when a cardmaster commits a crime? They end up wanted by the guild and locked up in Morgen's special prison.
+カードを取り上げたうえで、\n強いカードマスターの看守たちが24時間見張っているの。\n凶悪な犯罪者だって逃げだすことはできないわ。,They take away your cards and strong cardmaster guards watch over you 24/7. Even the most notorious criminals can't escape.
+看守の一人、Cランクのヤツメさまはとってもカッコいいんだから。\nあのクールな瞳。わたしあの人のファンなの。\n無敗のルーキーのあなたにだってきっとヤツメさまは負けないわよ。,One of the guards; a C-rank master named Yatsume; is really cool. Those intense eyes of hers... I'm a big fan. Even you wouldn't stand a chance against her; undefeated rookie!
```
Even you -The Undefeated Rookie- wouldn't stand a chance against her!

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+カードマスターが犯罪をしたらどうなると思う？\nギルドで指名手配されて、\nモルゲンの特殊牢獄に入れられるのよ。,What do you think happens when a cardmaster commits a crime? They end up wanted by the guild and locked up in Morgen's special prison.
+カードを取り上げたうえで、\n強いカードマスターの看守たちが24時間見張っているの。\n凶悪な犯罪者だって逃げだすことはできないわ。,They take away your cards and strong cardmaster guards watch over you 24/7. Even the most notorious criminals can't escape.
+看守の一人、Cランクのヤツメさまはとってもカッコいいんだから。\nあのクールな瞳。わたしあの人のファンなの。\n無敗のルーキーのあなたにだってきっとヤツメさまは負けないわよ。,One of the guards; a C-rank master named Yatsume; is really cool. Those intense eyes of hers... I'm a big fan. Even you wouldn't stand a chance against her; undefeated rookie!
+なあ聞いてくれよ。\n俺の応援してた男のCランクカードマスターがこの間ぼろ負けしてよ、\nついに降格しちまったんだ。,Hey; listen up! The C-rank cardmaster I was cheering for got completely crushed the other day and ended up getting demoted.
```
I was rooting for got completely crushed the other day; and ended up getting demoted too.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -52,321 +52,321 @@ Place/Colosseum,
+なあ聞いてくれよ。\n俺の応援してた男のCランクカードマスターがこの間ぼろ負けしてよ、\nついに降格しちまったんだ。,Hey; listen up! The C-rank cardmaster I was cheering for got completely crushed the other day and ended up getting demoted.
+女どもの誘惑に負けないけっこう強いマスターだったんだぜ？\nだけど、一回の試合では我慢できても、\n何回も戦っているうちにとうとう耐え切れなくなったみたいでよ。,He was a pretty strong cardmaster who wasn't swayed by the women's temptations; you know? But while he might've been able to hold out in one match; it seems after back-to-back fights he just couldn't stand it anymore.
+一回こっぴどく負けてからはすっかり快楽に弱くなっちまって…。\nそこからはいくらやっても女マスターに勝てなくなっちまった。,After losing badly just once; he became totally weak to pleasure attacks... From then on; no matter how hard he tried; he couldn't win against the female masters anymore.
+一度でも女モンスターたちの気持ちよさ味わうと、\n快感に抗えなくなっちまうのかな。\nアンタは負けないでくれよ、無敗のルーキー。,Once you experience the pleasure of a female monster even once; you can't resist the ecstasy anymore. So please; undefeated rookie; don't lose.
```
This sentence feels a little unnatural... hmm...

HotHeadEmperor replied:
Tried something

Giratena reviewed (changes requested):
Hey @HotHeadEmperor!

A respectable effort for such a large file! I did reach 30 comments on this file so I'm going to release what I have early so you can go ahead and take another look at it ^^

I spotted a lot of stiff sentences in this PR! Feel free to take a look again from an English stand point and see what you can find.

HotHeadEmperor committed:
Update Colosseum.txt

HotHeadEmperor committed:
Update Colosseum.txt

HotHeadEmperor committed:
Merge remote-tracking branch 'upstream/develop' into Arena-Location-Translation

HotHeadEmperor committed:
Revert "Update Colosseum.txt"

This reverts commit c8093833075bf1ac06bd3fd24561b607aab1db70.

HotHeadEmperor committed:
Revert "Merge remote-tracking branch 'upstream/develop' into Arena-Location-Translation"

This reverts commit 40278920e516a2a7d3fea74feae9403faa822323, reversing
changes made to c8093833075bf1ac06bd3fd24561b607aab1db70.

HotHeadEmperor committed:
Reapply "Merge remote-tracking branch 'upstream/develop' into Arena-Location-Translation"

This reverts commit d85a7139af206165b0449075e5b4f3b9688fdb5a.

HotHeadEmperor committed:
Update Colosseum.txt

HotHeadEmperor committed:
Update Colosseum.txt

Line 317 needs a check

HotHeadEmperor committed:
Update Colosseum.txt

HotHeadEmperor committed:
Update Colosseum.txt

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -58,15 +58,15 @@ Place/Colosseum,
 ・・・。,...
-ありがとうございます。貴方がDランクであることを確認しました。闘技場への参加を承認します。,Thank you very much. After confirming your rank is D; I gladly approve your participation in the colosseum.
-再度話しかけていただければ、本日からでも対戦を組むことができますので、準備ができたらお声がけくださって構いませんよ。,If you speak to me again; we can arrange a match starting today; so feel free to let me know when you're ready.
+ありがとうございます。貴方がDランクであることを確認しました。闘技場への参加を承認します。,Thank you so much. After confirming your rank; I'm happy to say your application for the colosseum has been approved.
```
Much better!

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -58,15 +58,15 @@ Place/Colosseum,
-ありがとうございます。貴方がDランクであることを確認しました。闘技場への参加を承認します。,Thank you very much. After confirming your rank is D; I gladly approve your participation in the colosseum.
-再度話しかけていただければ、本日からでも対戦を組むことができますので、準備ができたらお声がけくださって構いませんよ。,If you speak to me again; we can arrange a match starting today; so feel free to let me know when you're ready.
+ありがとうございます。貴方がDランクであることを確認しました。闘技場への参加を承認します。,Thank you so much. After confirming your rank; I'm happy to say your application for the colosseum has been approved.
+再度話しかけていただければ、本日からでも対戦を組むことができますので、準備ができたらお声がけくださって構いませんよ。,Feel free to speak to me again when you're ready and I'll happily arrange a match for you today.
```
Very good!

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -58,15 +58,15 @@ Place/Colosseum,
-国中から集まってきたカードマスターたちの火花散らす戦い…。\nいやぁ、たまらないね。,Cardmasters from all over the country gather and have fierce battles... Ah; the excitment of it all is just intoxicating; don't you think?
-でもよ、最近は女マスターばっかり勝っててつまらねえ。\nもっと剣と剣がぶつかるような戦いを、前みたいに見たいもんだ。\n男としてあんたには期待してるぜ！,But you know; lately it's only been female masters winning; and it's actually starting to get pretty boring. I want to see battles where swords clash like they used to. So as a guy; I'm counting on you!
-闘技場って、中にいるだけで何だかワクワクしてくるよな。,The colosseum... Just being here makes you a bit excited; doesn't it?
+国中から集まってきたカードマスターたちの火花散らす戦い…。\nいやぁ、たまらないね。,Cardmasters from all over the country gather to battle it out... Ah; the excitment of it all. I can hardly contain myself
```
Add ! At the end here.

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -106,166 +106,166 @@ Bランクまで上がった学生は国中でこの三人だけだからな。\
 最近どうも王都の方がきなくさいらしいな。\nギルドマスターや高ランクのマスターが盛んに行き来していて、\nそろそろ大きな戦でもあるんじゃないかって話だぜ。,Recently; it feels like something's brewing in the royal capital. Guild masters and high-ranking cardmasters have been coming and going frequently; and there's talk that a major conflict might be on the horizon.
-王都とモルゲンには強力な結界があって、\nサキュバスたちも簡単には手を出せねえ。\nそれでしばらく膠着状態だったんだが、いよいよ動き出すのかもな。,The royal capital and Morgen have powerful barriers; so even the succubi can't easily make a move. That's why the war's been in a stalemate for a while; but it might finally be starting to move again.
-まあ、モルゲンは魔族領からは遠いから\n直接戦いになったりはしないだろうけどよ。\n王都に食い物が流れる分、値上げとかするかもしれねえな。,Well; since Morgen is pretty far from the demon territory; the fighting probably won't reach us. Still; since the royal capital's food supplies will be prioritized; prices might go up.
+王都とモルゲンには強力な結界があって、\nサキュバスたちも簡単には手を出せねえ。\nそれでしばらく膠着状態だったんだが、いよいよ動き出すのかもな。,The royal capital and Morgen have powerful barriers; so even the succubi can't easily make a move. It's left the war in a standstill for a while, but tensions might finally be starting to flare again.
```
, here

Flare again -> flare up again

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -106,166 +106,166 @@ Bランクまで上がった学生は国中でこの三人だけだからな。\
-カードを取り上げたうえで、\n強いカードマスターの看守たちが24時間見張っているの。\n凶悪な犯罪者だって逃げだすことはできないわ。,They take away your cards and strong cardmaster guards watch over you 24/7. Even the most notorious criminals can't escape.
-看守の一人、Cランクのヤツメさまはとってもカッコいいんだから。\nあのクールな瞳。わたしあの人のファンなの。\n無敗のルーキーのあなたにだってきっとヤツメさまは負けないわよ。,One of the guards; a C-rank master named Yatsume; is really cool. Those intense eyes of hers... I'm a big fan. Even you wouldn't stand a chance against her; undefeated rookie!
-なあ聞いてくれよ。\n俺の応援してた男のCランクカードマスターがこの間ぼろ負けしてよ、\nついに降格しちまったんだ。,Hey; listen up! The C-rank cardmaster I was cheering for got completely crushed the other day and ended up getting demoted.
+カードマスターが犯罪をしたらどうなると思う？\nギルドで指名手配されて、\nモルゲンの特殊牢獄に入れられるのよ。,What do you think happens when a cardmaster commits a crime? They end up on the guild's wanted list, and are locked up in Morgen's special prison.
```
, here

Giratena commented on CSV/Language/English/Place/Colosseum.txt:
```diff
@@ -106,166 +106,166 @@ Bランクまで上がった学生は国中でこの三人だけだからな。\
-コントロール能力さえ高ければ魔力消費を抑えられるのは\nメリットにあげられるのかもしれませんね。,If you can control the magical energy; I suppose being able to reduce magic power consumption could be considered an advantage; perhaps?
+このように理性を持たないカードは人を選ばず出ることから、\n先ほど紹介したような、理性のあるカードは前世のつながりがあるのでは\nという突飛な説が生まれたわけです。,It's the fact that non-sentient cards appear no matter who opens a pack that led to the outlandish theory I mentioned earlier that sentient cards might have connections to past lives.
+モンスターたちが所持している道具の中には、\n我々の生活の中ではあまり見られないものも多くあります。,Among the tools possessed by monsters; there are many that are rarely seen in our daily lives.
+例えば一部のモンスターが所持している「銃」がそうですね。\n火の魔法を込めて、小さな石の弾や鉄の弾を飛ばす、\n魔法の小型杖です。,For example; some monsters carry 'guns;' small; magical staves that shoot tiny stone or iron projectiles infused with fire magic.
```
'Guns;' small

HotHeadEmperor replied:
I don't think it'd be capitalized, since even if he's quoting what others call them, it's still not a proper noun.

Giratena replied:
Ah my apologies, the issue wasn't the capital letter (auto correct did that aha).

I think you meant to say this:
For example; some monsters carry 'guns;' small; magical staves that shoot tiny stone or iron projectiles infused with fire magic. 
->
For example; some monsters carry **'guns'—small;** magical staves that shoot tiny stone or iron projectiles infused with fire magic.

Giratena reviewed (commented):
Very nice coverage @HotHeadEmperor! I appreciate your second once over and I think you covered a lot of good stuff!

I will review this again, but I think overall it's looking much better and I think we should be able to get this in on the second review pass. Nice job!

HotHeadEmperor committed:
Update Colosseum.txt

HotHeadEmperor committed:
Update Colosseum.txt
